--- "Haunted House (Gottlieb 1982) VPW 2.0.vbs.original"	2023-10-30 10:53:14.330817710 +0100
+++ "Haunted House (Gottlieb 1982) VPW 2.0.vbs"	2023-10-30 10:52:38.041888676 +0100
@@ -1467,7 +1467,7 @@
 		Case 2:
 		Case 3:	SoundSaucerKick 1,sw31
 		'Case 4: sw31.Kickxyz 0,60,1.50,-7,-1,30
-		Case 4: sw31.Kickxyz (Rnd()-0.5)*20,50 + (Rnd()*5),1.50,-7,-1,20
+		Case 4: sw31.Kickxyz 20*(Rnd()-0.5),50 + (Rnd()*5),1.50,-7,-1,20
 		Case 5:	
 		Case 6:
 		Case 7:
