--- Phantom Of The Opera (Data East 1990) v1.24.vbs.original
+++ Phantom Of The Opera (Data East 1990) v1.24.vbs
@@ -319,4 +319,4 @@
-		'.Games(cGameName).Settings.Value("sound") = 1 	' Uncomment for Standalone compatability - comment out matching statements
-		'.Games(cGameName).Settings.Value("rol") = 0	' Uncomment for Standalone compatability - comment out matching statements
-		.Settings.value("sound") = 1  '- Test table sounds...  disables ROM sounds
-        .Settings.Value("rol") = 0
+		.Games(cGameName).Settings.Value("sound") = 1 	' Uncomment for Standalone compatability - comment out matching statements
+		.Games(cGameName).Settings.Value("rol") = 0	' Uncomment for Standalone compatability - comment out matching statements
+		'.Settings.value("sound") = 1  '- Test table sounds...  disables ROM sounds
+        '.Settings.Value("rol") = 0
