--- UT99CTF_GE_2.3.vbs.original
+++ UT99CTF_GE_2.3.vbs
@@ -1206,4 +1206,4 @@
-	scene.AddActor FlexDMD.NewImage("Arrow0", FlexPath & "SNA/Arrowup.png" )
-	scene.AddActor FlexDMD.NewImage("Arrow1", FlexPath & "SNA/Arrowright.png" )
-	scene.AddActor FlexDMD.NewImage("Arrow2", FlexPath & "SNA/Arrowdown.png" )
-	scene.AddActor FlexDMD.NewImage("Arrow3", FlexPath & "SNA/Arrowleft.png" )
+	scene.AddActor FlexDMD.NewImage("Arrow0", FlexPath & "SNA/arrowup.png" )
+	scene.AddActor FlexDMD.NewImage("Arrow1", FlexPath & "SNA/arrowright.png" )
+	scene.AddActor FlexDMD.NewImage("Arrow2", FlexPath & "SNA/arrowdown.png" )
+	scene.AddActor FlexDMD.NewImage("Arrow3", FlexPath & "SNA/arrowleft.png" )
@@ -1227 +1227 @@
-'	scene.AddActor FlexDMD.NewImage("Gameover", FlexPath & "SNA/gameover.png" )
+'	scene.AddActor FlexDMD.NewImage("Gameover", FlexPath & "SNA/Gameover.png" )
@@ -1237 +1237 @@
-			Set title = FlexDMD.NewImage("Gameover", flexpath & "SNA/gameover.png" )
+			Set title = FlexDMD.NewImage("Gameover", flexpath & "SNA/Gameover.png" )
@@ -1273 +1273 @@
-			Set title = FlexDMD.NewImage("Highscore", flexpath & "SNA/highscore.png" )
+			Set title = FlexDMD.NewImage("Highscore", flexpath & "SNA/Highscore.png" )
@@ -1550 +1550 @@
-		Playmusic "Unrealtournament/UT-Champions.mp3"
+		Playmusic "UnrealTournament/UT-Champions.mp3"
@@ -1554 +1554 @@
-		Playmusic "Unrealtournament/UT-Menu.mp3"
+		Playmusic "UnrealTournament/UT-Menu.mp3"
@@ -11821 +11821 @@
-		Playmusic "Unrealtournament/UT-Menu.mp3"
+		Playmusic "UnrealTournament/UT-Menu.mp3"
@@ -11831 +11831 @@
-	PlayMusic "Unrealtournament/UT" & int(rnd(1)*13)+1 & ".mp3"
+	PlayMusic "UnrealTournament/UT" & int(rnd(1)*13)+1 & ".mp3"
@@ -12452 +12452 @@
-			If Frame = 330 Then	Playmusic "Unrealtournament/UT-Title.mp3" :	MusicVolume = Music_Volume
+			If Frame = 330 Then	Playmusic "UnrealTournament/UT-Title.mp3" :	MusicVolume = Music_Volume
