--- Diablo Pinball v4.3.vbs.original
+++ Diablo Pinball v4.3.vbs
@@ -2347,11 +2347,11 @@
-    Dim WshShell:Set WshShell = CreateObject("WScript.Shell")
-    ' Read the current Values
-    On Error Resume Next
-    DMDOldColor = WshShell.RegRead("HKCU\Software\UltraDMD\color")
-    DMDOldFullColor = WshShell.RegRead("HKCU\Software\UltraDMD\fullcolor")
-    If DMDOldFullColor = "" Then DMDOldFullColor = "False"
-    ' Set table Values
-    WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor", "True", "REG_SZ"
-    WshShell.RegWrite "HKCU\Software\UltraDMD\color", "Tomato", "REG_SZ"
-    WshShell.RegWrite "HKCU\Software\UltraDMD\hideVDMD", "False", "REG_SZ"
-    On Error Goto 0
+    ' Dim WshShell:Set WshShell = CreateObject("WScript.Shell")
+    ' ' Read the current Values
+    ' On Error Resume Next
+    ' DMDOldColor = WshShell.RegRead("HKCU\Software\UltraDMD\color")
+    ' DMDOldFullColor = WshShell.RegRead("HKCU\Software\UltraDMD\fullcolor")
+    ' If DMDOldFullColor = "" Then DMDOldFullColor = "False"
+    ' ' Set table Values
+    ' WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor", "True", "REG_SZ"
+    ' WshShell.RegWrite "HKCU\Software\UltraDMD\color", "Tomato", "REG_SZ"
+    ' WshShell.RegWrite "HKCU\Software\UltraDMD\hideVDMD", "False", "REG_SZ"
+    ' On Error Goto 0
@@ -2362,3 +2362,3 @@
-    Dim WshShell:Set WshShell = CreateObject("WScript.Shell")
-    WshShell.RegWrite "HKCU\Software\UltraDMD\color", DMDOldColor, "REG_SZ"
-    WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor", DMDOldFullColor, "REG_SZ"
+    ' Dim WshShell:Set WshShell = CreateObject("WScript.Shell")
+    ' WshShell.RegWrite "HKCU\Software\UltraDMD\color", DMDOldColor, "REG_SZ"
+    ' WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor", DMDOldFullColor, "REG_SZ"
