--- ZoneFuryVPX.vbs.original	2023-09-04 11:43:02
+++ ZoneFuryVPX.vbs	2023-09-04 10:51:30
@@ -626,8 +626,8 @@
 				Controller.B2SSetCredits (7*(Player-1)+1),128+SegValueB2S(Digit)
 			End Select
 		Else
-			Controller.B2SSetCredits (7*(Player-1))+6,SegValueB2S(0)
- 			Controller.B2SSetCredits (7*(Player-1))+7,SegValueB2S(0)
+			Controller.B2SSetCredits ((7*(Player-1))+6),SegValueB2S(0)
+ 			Controller.B2SSetCredits ((7*(Player-1))+7),SegValueB2S(0)
 		End If
 	End If
 
@@ -670,7 +670,7 @@
 	For x = 0 to 3
  		For y = 0 to 6
 			ScoreDisplay(x,y).SetValue 0
-			If B2SOn Then Controller.B2SSetCredits (7*x)+(y+1),0
+			If B2SOn Then Controller.B2SSetCredits ((7*x)+(y+1)),0
 		Next
 	Next
  
@@ -683,7 +683,7 @@
 
 	For y = 0 to 6
 		ScoreDisplay(Player-1,y).SetValue 0
-		If B2SOn Then Controller.B2SSetCredits 7*(Player-1)+(y+1),0
+		If B2SOn Then Controller.B2SSetCredits (7*(Player-1)+(y+1)),0
 	Next
  
 End Sub
