--- Streets of Rage (TBA 2018).vbs.original
+++ Streets of Rage (TBA 2018).vbs
@@ -75 +75 @@
-GetDMDColor
+'GetDMDColor
@@ -4079 +4079 @@
-				    DisplayB2SText (Score(1)) & String(10 - Len(PosInTable), " ") & "HIGH SCORE <" & PosInTable & Suffix & ">"
+				    DisplayB2SText "" & (Score(1)) & String(10 - Len(PosInTable), " ") & "HIGH SCORE <" & PosInTable & Suffix & ">"
@@ -4082 +4082 @@
-			    	DisplayB2SText (Score(1)) & String(16 - Len(Score(1)) - Len(PosInTable), " ") & "HIGH SCORE <" & PosInTable & Suffix & ">"
+			    	DisplayB2SText "" & (Score(1)) & String(16 - Len(Score(1)) - Len(PosInTable), " ") & "HIGH SCORE <" & PosInTable & Suffix & ">"
@@ -4093 +4093 @@
-                    DisplayB2SText (Score(1)) & "                GAME OVER"
+                    DisplayB2SText "" & (Score(1)) & "                GAME OVER"
@@ -4096 +4096 @@
-				    DisplayB2SText (Score(1)) & String(22 - Len(Score(1)), " ") & "GAME OVER"
+				    DisplayB2SText "" & (Score(1)) & String(22 - Len(Score(1)), " ") & "GAME OVER"
@@ -4232 +4232 @@
-                DisplayB2SText (POWBonus) & " SOR     " & String(lenPOWBase, " ") & "     " & String(lenPOWScore, " ") & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
+                DisplayB2SText POWBonus & " SOR     " & String(lenPOWBase, " ") & "     " & String(lenPOWScore, " ") & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
@@ -4239 +4239 @@
-                DisplayB2SText (POWBonus) & " SOR  X  " & String(lenPOWBase, " ") & "     " & String(lenPOWScore, " ") & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
+                DisplayB2SText POWBonus & " SOR  X  " & String(lenPOWBase, " ") & "     " & String(lenPOWScore, " ") & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
@@ -4246 +4246 @@
-                DisplayB2SText (POWBonus) & " SOR  X  " & FormatScore(nvR2 * 1000) & "     " & String(lenPOWScore, " ") & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
+                DisplayB2SText POWBonus & " SOR  X  " & FormatScore(nvR2 * 1000) & "     " & String(lenPOWScore, " ") & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
@@ -4253 +4253 @@
-                DisplayB2SText (POWBonus) & " SOR  X  " & FormatScore(nvR2 * 1000) & "  =  " & String(lenPOWScore, " ") & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
+                DisplayB2SText POWBonus & " SOR  X  " & FormatScore(nvR2 * 1000) & "  =  " & String(lenPOWScore, " ") & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
@@ -4260 +4260 @@
-                DisplayB2SText (POWBonus) & " SOR  X  " & FormatScore(nvR2 * 1000) & "  =  " & FormatScore(1000 * nvR2 * POWBonus) & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
+                DisplayB2SText POWBonus & " SOR  X  " & FormatScore(nvR2 * 1000) & "  =  " & FormatScore(1000 * nvR2 * POWBonus) & String(Int((18 - lenPOWBonus - lenPOWBase - lenPOWScore) / 2), " ")
@@ -5427 +5427 @@
-                    DisplayB2SText (cstr(FormatScore(Score(1)))) & "        REMAINING WEAPONS " & Right("0" & RemainingJets, 2)
+                    DisplayB2SText cstr(FormatScore(Score(1))) & "        REMAINING WEAPONS " & Right("0" & RemainingJets, 2)
@@ -5430 +5430 @@
-                    DisplayB2SText (cstr(FormatScore(Score(1)))) & String(14 - Len(Score(1)), " ") & "REMAINING WEAPONS " & Right("0" & RemainingJets, 2)
+                    DisplayB2SText cstr(FormatScore(Score(1))) & String(14 - Len(Score(1)), " ") & "REMAINING WEAPONS " & Right("0" & RemainingJets, 2)
@@ -5436 +5436 @@
-                    DisplayB2SText (cstr(FormatScore(Score(1))))
+                    DisplayB2SText cstr(FormatScore(Score(1)))
@@ -5439 +5439 @@
-                    DisplayB2SText (cstr(FormatScore(Score(1)))) & String(16 - Len(Score(1)), " ") & "REMAINING SOR " & RemainingPOW
+                    DisplayB2SText cstr(FormatScore(Score(1))) & String(16 - Len(Score(1)), " ") & "REMAINING SOR " & RemainingPOW
@@ -5445 +5445 @@
-                    DisplayB2SText (cstr(FormatScore(Score(1)))) & "         BOSS ENERGY " & RemainingAllen
+                    DisplayB2SText cstr(FormatScore(Score(1))) & "         BOSS ENERGY " & RemainingAllen
@@ -5448 +5448 @@
-                    DisplayB2SText (cstr(FormatScore(Score(1)))) & String(15 - Len(Score(1)), " ") & "BOSS ENERGY " & RemainingAllen
+                    DisplayB2SText cstr(FormatScore(Score(1))) & String(15 - Len(Score(1)), " ") & "BOSS ENERGY " & RemainingAllen
@@ -6612 +6612 @@
-    DisplayB2SText (POWBonus) & " SOR SAVED   " & String(Len(POWRank(POWRankRnd)), " ") & " " & String(Len(POWName(POWNamePtr)), " ") & String(Int((18 - Len(POWBonus) - Len(POWRank(POWRankRnd)) - Len(POWName(POWNamePtr))) / 2), " ")
+    DisplayB2SText POWBonus & " SOR SAVED   " & String(Len(POWRank(POWRankRnd)), " ") & " " & String(Len(POWName(POWNamePtr)), " ") & String(Int((18 - Len(POWBonus) - Len(POWRank(POWRankRnd)) - Len(POWName(POWNamePtr))) / 2), " ")
@@ -7404 +7404 @@
-	 DisplayB2SText (cstr(Score(1))) & "" & "                   BALL " & Ball
+	 DisplayB2SText cstr(Score(1)) & "" & "                   BALL " & Ball
@@ -7406 +7406 @@
-	 DisplayB2SText (cstr(Score(1))) & (Score(1)) & String(32 - Len(Score(1)), " ") 
+	 DisplayB2SText cstr(Score(1)) & (Score(1)) & String(32 - Len(Score(1)), " ") 
