--- Aladdin's Castle (Bally 1976) - DOZER - MJR_1.01.vbs.original
+++ Aladdin's Castle (Bally 1976) - DOZER - MJR_1.01.vbs
@@ -628 +628,4 @@
-'******* Key Down *********************************************************************************************************Dim gxx
+
+'******* Key Down *********************************************************************************************************
+Dim gxx
+
@@ -630 +633,2 @@
-	If  Keycode = PlungerKey Then       'JP's plunger stuff.		Plunger.PullBack                'Pull back the plunger. 
+	If  Keycode = PlungerKey Then       'JP's plunger stuff.
+		Plunger.PullBack                'Pull back the plunger. 
@@ -633 +637,3 @@
-        End If	End If    	If  Keycode = LeftFlipperKey And InProgress = True And Tilt=False Then
+        End If
+	End If    
+	If  Keycode = LeftFlipperKey And InProgress = True And Tilt=False Then
@@ -638 +644,2 @@
-		LeftFlipper.RotateToEnd         'If the above conditions are present then flipper goes up.        PlaySoundat SoundFXDOF("flipperup",128,DOFOn,DOFFlippers), leftflipper
+		LeftFlipper.RotateToEnd         'If the above conditions are present then flipper goes up.
+        PlaySoundat SoundFXDOF("flipperup",128,DOFOn,DOFFlippers), leftflipper
@@ -646 +653,3 @@
-        End If	End If    	If  Keycode = RightFlipperKey And InProgress = True And Tilt=False Then
+        End If
+	End If    
+	If  Keycode = RightFlipperKey And InProgress = True And Tilt=False Then
@@ -651 +660,2 @@
-		RightFlipper1.RotateToEnd       'If the above conditions are present then flipper goes up.		PlaySoundat SoundFXDOF("flipperup",129,DOFOn,DOFFlippers), rightflipper
+		RightFlipper1.RotateToEnd       'If the above conditions are present then flipper goes up.
+		PlaySoundat SoundFXDOF("flipperup",129,DOFOn,DOFFlippers), rightflipper
@@ -659 +669,5 @@
-        End If	End If	If  Keycode = AddCreditKey Then 		Credits=credits + 1             'Add a credit.		MaxCredits                      'Call the max credits sub to check for maximum credits. 
+        End If
+	End If
+	If  Keycode = AddCreditKey Then 
+		Credits=credits + 1             'Add a credit.
+		MaxCredits                      'Call the max credits sub to check for maximum credits. 
@@ -664 +678,3 @@
-        End If		' If  InProgress= False Then TextBox.Text = "Press Start"   'If the game is over then show Press Start.		PlaySound "coinin"               'I wish I had a quarter for every time I've heard this sound. Amen, Bob!
+        End If
+		' If  InProgress= False Then TextBox.Text = "Press Start"   'If the game is over then show Press Start.
+		PlaySound "coinin"               'I wish I had a quarter for every time I've heard this sound. Amen, Bob!
@@ -668,4 +684,13 @@
-	End If   	If  Keycode = LeftTiltKey Then      'Left shake.		Nudge 90, 2                     'Degree of shake and strength.
-		BumpIt                          'Check for tilt	End If    	If  Keycode = RightTiltKey Then     'Right shake.		Nudge 270, 2                    'Degree of shake and strength.
-		BumpIt                          'Check for tilt	End If    	If  Keycode = CenterTiltKey Then    'Center shake.		Nudge 0, 2                      'Degree of shake and strength.
-		BumpIt                          'Check for tilt	End If
+	End If   
+	If  Keycode = LeftTiltKey Then      'Left shake.
+		Nudge 90, 2                     'Degree of shake and strength.
+		BumpIt                          'Check for tilt
+	End If    
+	If  Keycode = RightTiltKey Then     'Right shake.
+		Nudge 270, 2                    'Degree of shake and strength.
+		BumpIt                          'Check for tilt
+	End If    
+	If  Keycode = CenterTiltKey Then    'Center shake.
+		Nudge 0, 2                      'Degree of shake and strength.
+		BumpIt                          'Check for tilt
+	End If
@@ -846 +871,7 @@
-'*********** Key Up ******************************************************************************************************* Sub Table1_KeyUp(ByVal keycode)     'This is what happens when you release a key.	If  keycode = PlungerKey Then       'JP's Plunger stuff.		Plunger.Fire                    'Fire the plunger.		Playsoundat "Plungerrelease", Screw61
+
+'*********** Key Up *******************************************************************************************************
+ 
+Sub Table1_KeyUp(ByVal keycode)     'This is what happens when you release a key.
+	If  keycode = PlungerKey Then       'JP's Plunger stuff.
+		Plunger.Fire                    'Fire the plunger.
+		Playsoundat "Plungerrelease", Screw61
@@ -849 +880,3 @@
-        End If	End If   	If  Keycode = LeftFlipperKey then
+        End If
+	End If   
+	If  Keycode = LeftFlipperKey then
@@ -854 +887,2 @@
-            End If			LeftFlipper.RotateToStart       'If the above conditions are true the flipper goes down.
+            End If
+			LeftFlipper.RotateToStart       'If the above conditions are true the flipper goes down.
@@ -857 +891,2 @@
-	End If    	If  Keycode = RightFlipperKey then
+	End If    
+	If  Keycode = RightFlipperKey then
@@ -862 +897,2 @@
-            End If			RightFlipper.RotateToStart      'If the above conditions are true the flipper goes down.
+            End If
+			RightFlipper.RotateToStart      'If the above conditions are true the flipper goes down.
@@ -864 +900,2 @@
-			DOF 129, 0			PlaySoundat SoundFXDOF("flipperdown",129,DOFOff,DOFFlippers), RightFlipper
+			DOF 129, 0
+			PlaySoundat SoundFXDOF("flipperdown",129,DOFOff,DOFFlippers), RightFlipper
@@ -886 +923,2 @@
-	Credits=credits + 1             'Add a credit.    MaxCredits                      'Call the max credits sub to check for maximum credits. 
+	Credits=credits + 1             'Add a credit.
+    MaxCredits                      'Call the max credits sub to check for maximum credits. 
@@ -917,0 +956 @@
+
@@ -918,0 +958 @@
+ 
@@ -939 +979,2 @@
-            End If  			Players = Players + 1           'Add a player.
+            End If  
+			Players = Players + 1           'Add a player.
@@ -956 +997,4 @@
-Sub NewGame                          ' Start game, kickass I found a quarter!    Ball=1                           ' Ball 1.	For Each Obj In Toplights:Obj.State=1:Next    ' Top lanes are all lit at the start of each ball
+
+Sub NewGame                          ' Start game, kickass I found a quarter!
+    Ball=1                           ' Ball 1.
+	For Each Obj In Toplights:Obj.State=1:Next    ' Top lanes are all lit at the start of each ball
@@ -980 +1024,2 @@
-	next    DelayTimer1.Enabled=true
+	next
+    DelayTimer1.Enabled=true
@@ -1015,0 +1061 @@
+ 
@@ -1024 +1070,4 @@
- '*********Sound Effects**************************************************************************************************                                       'Use these for your sound effects like ball rolling, etc.
+ 
+'*********Sound Effects**************************************************************************************************
+                                       'Use these for your sound effects like ball rolling, etc.
+
@@ -1094 +1143,2 @@
-End Sub'******************* Bumpers **********************************************************************************************
+End Sub
+'******************* Bumpers **********************************************************************************************
@@ -1117,0 +1168 @@
+
@@ -1121,2 +1172,5 @@
-		' and tilt if we reach 3 bumps in within too short a time span.		Tiltcount = Tiltcount + 1       ' add 1 to the tiltcount while starting the timer.		TiltTimer.Enabled = True
-	End If	If  Tiltcount > 2 Then              
+		' and tilt if we reach 3 bumps in within too short a time span.
+		Tiltcount = Tiltcount + 1       ' add 1 to the tiltcount while starting the timer.
+		TiltTimer.Enabled = True
+	End If
+	If  Tiltcount > 2 Then              
@@ -1124 +1178,2 @@
-		' time window.  Tilt the game.		Tilt = True
+		' time window.  Tilt the game.
+		Tilt = True
@@ -1128 +1183,2 @@
-	End If End Sub
+	End If 
+End Sub
@@ -1188,2 +1244,11 @@
-End Sub   Sub TiltTimer_Timer    TiltTimer.Enabled = False       'Turn off/reset the timer.	If  Tiltcount=3 Then                'If the tilt count reches 3 then the game is tilted,
-        Tilt = True                     'We have tilted the game.	Else                             		Tiltcount = 0                   'Else tilt count is 0 and reset the tilt timer.	End IfEnd Sub
+End Sub
+   
+Sub TiltTimer_Timer
+    TiltTimer.Enabled = False       'Turn off/reset the timer.
+	If  Tiltcount=3 Then                'If the tilt count reches 3 then the game is tilted,
+        Tilt = True                     'We have tilted the game.
+	Else                             
+		Tiltcount = 0                   'Else tilt count is 0 and reset the tilt timer.
+	End If
+End Sub
+ 	 
@@ -1316,0 +1382 @@
+
@@ -1649 +1715,3 @@
-	End IfEnd Sub
+	End If
+End Sub
+
@@ -1654 +1722,2 @@
-			DOF 126, 2			AddScore100                    'Add the score.
+			DOF 126, 2
+			AddScore100                    'Add the score.
@@ -1659 +1728,3 @@
-	End IfEnd Sub 
+	End If
+End Sub
+ 
@@ -1663 +1734,2 @@
-		ScoreMotorAdd 300           ' score 300 on the score motor		DOF 115, 1                  ' fire the DOF effect
+		ScoreMotorAdd 300           ' score 300 on the score motor
+		DOF 115, 1                  ' fire the DOF effect
@@ -1665 +1737,2 @@
-	End IfEnd Sub 
+	End If
+End Sub 
@@ -1694 +1767,2 @@
-		If  Sidelight1.State=1 Then			ScoreAladdinsAlley
+		If  Sidelight1.State=1 Then
+			ScoreAladdinsAlley
@@ -1819,2 +1893,7 @@
-' Start a new ballSub AddBall()	If Shootagain.State=1 Then         'If the Shoot Again light is on then subtract a player. The Bonus Ended sub will add 		Player=Player-1 'a player to go to the next player, subtracting a player counteracts this, keeping the same player.
-        If B2SOn Then		Controller.B2ssetshootagain 36,0
+
+' Start a new ball
+Sub AddBall()
+	If Shootagain.State=1 Then         'If the Shoot Again light is on then subtract a player. The Bonus Ended sub will add 
+		Player=Player-1 'a player to go to the next player, subtracting a player counteracts this, keeping the same player.
+        If B2SOn Then
+		Controller.B2ssetshootagain 36,0
@@ -1825,2 +1904,7 @@
-	End IfEnd Sub
-' When the ball drains, start the bonus Sub Drain_Hit()                     'Another one bites the dust.    Drain.DestroyBall               'Destroy the ball when it hits the drain.    PlaySoundat "Drain5", drain             'Play the sound.
+	End If
+End Sub
+
+' When the ball drains, start the bonus 
+Sub Drain_Hit()                     'Another one bites the dust.
+    Drain.DestroyBall               'Destroy the ball when it hits the drain.
+    PlaySoundat "Drain5", drain             'Play the sound.
@@ -1827,0 +1912 @@
+
@@ -1836 +1921,2 @@
-' When the bonus ends, proceed to the next ballSub BonusEnded()
+' When the bonus ends, proceed to the next ball
+Sub BonusEnded()
@@ -1838 +1924,4 @@
-    Player=Player+1                 'Go to the next player.    BonusCounter = 1000             'Reset the bonus to 1000 for the next ball.    Light1k.State = 1               'Turn on the 1000 light for the next ball.    AlleyLevel = 0                  'Reset the Aladdin's Alley progressive score to the first level
+    Player=Player+1                 'Go to the next player.
+    BonusCounter = 1000             'Reset the bonus to 1000 for the next ball.
+    Light1k.State = 1               'Turn on the 1000 light for the next ball.
+    AlleyLevel = 0                  'Reset the Aladdin's Alley progressive score to the first level
@@ -1849 +1938,3 @@
-	End If    	If  Ball <= OptBallsPerGame Then   'Is ball played less than total Balls or is the Game Over.		DelayTimer2.Enabled=True
+	End If    
+	If  Ball <= OptBallsPerGame Then   'Is ball played less than total Balls or is the Game Over.
+		DelayTimer2.Enabled=True
@@ -1851 +1942,2 @@
-		For Each Obj In Toplights:Obj.State=1:Next        ' top lanes start each ball ON		For Each Obj In Sidelights: Obj.State=0:Next
+		For Each Obj In Toplights:Obj.State=1:Next        ' top lanes start each ball ON
+		For Each Obj In Sidelights: Obj.State=0:Next
@@ -1873,2 +1965,6 @@
-	Else                                'Sorry, but the game is now over.		GameOver                        'Go to game over to finish up.
-		Light1k.State=0	End IfEnd Sub
+
+	Else                                'Sorry, but the game is now over.
+		GameOver                        'Go to game over to finish up.
+		Light1k.State=0
+	End If
+End Sub
@@ -1884,2 +1980,7 @@
-End Sub '************ Bonus Routine ************************************************************************************************
-Dim Obj,BonusCounter,MultiplierCounter,Tens,Ones Sub AdvanceBonus()
+End Sub
+ 
+'************ Bonus Routine ************************************************************************************************
+
+Dim Obj,BonusCounter,MultiplierCounter,Tens,Ones
+ 
+Sub AdvanceBonus()
@@ -1887 +1988,4 @@
-	If  Bonuscounter >15000 Then Bonuscounter=15000	ShowBonusLightsEnd Sub 
+	If  Bonuscounter >15000 Then Bonuscounter=15000
+	ShowBonusLights
+End Sub
+ 
@@ -1894,0 +1999 @@
+
@@ -1897 +2002,2 @@
-	' turn off all of the lights to start fresh	For Each Obj In BonusLights:Obj.State = 0:Next
+	' turn off all of the lights to start fresh
+	For Each Obj In BonusLights:Obj.State = 0:Next
@@ -1899 +2005,3 @@
-	' Figure the multiple of 1000 and 10000 for the current bonus value.	Tens = ((Bonuscounter + 500) \ 10000) MOD 10   ' Get the multiple of 10,000	Ones = ((Bonuscounter + 500) \ 1000) MOD 10    ' Get the multiple if 1,000
+	' Figure the multiple of 1000 and 10000 for the current bonus value.
+	Tens = ((Bonuscounter + 500) \ 10000) MOD 10   ' Get the multiple of 10,000
+	Ones = ((Bonuscounter + 500) \ 1000) MOD 10    ' Get the multiple if 1,000
@@ -1901 +2009,4 @@
-	BonusTensLights(Tens).State = 1                ' Turn on the suitable 10000s lightEnd Sub  Sub BonusTimer_Timer 
+	BonusTensLights(Tens).State = 1                ' Turn on the suitable 10000s light
+End Sub
+  
+Sub BonusTimer_Timer 
@@ -1956,0 +2068 @@
+
@@ -1957,0 +2070 @@
+
@@ -1967 +2080,2 @@
-	LeftFlipper.RotateToStart       'If the above conditions are true the flipper goes down.    PlaySoundAt SoundFXDOF("FlipperDown",128,DOFOff,DOFFlippers), LeftFlipper
+	LeftFlipper.RotateToStart       'If the above conditions are true the flipper goes down.
+    PlaySoundAt SoundFXDOF("FlipperDown",128,DOFOff,DOFFlippers), LeftFlipper
@@ -1969 +2083,2 @@
-	RightFlipper1.RotateToStart     'If the above conditions are true the flipper goes down.    PlaySoundAt SoundFXDOF("FlipperDown",129,DOFOff,DOFFlippers), RightFlipper 
+	RightFlipper1.RotateToStart     'If the above conditions are true the flipper goes down.
+    PlaySoundAt SoundFXDOF("FlipperDown",129,DOFOff,DOFFlippers), RightFlipper 
@@ -2432 +2547 @@
-			HSInitStr = Left(HSInitStr, Len(HSInitStr)-1)			
+			HSInitStr = Left(HSInitStr, (Len(HSInitStr)-1))			
@@ -2547 +2662 @@
-				ele(2).SetValue (ele(3) + val)*2
+				ele(2).SetValue 2*(ele(3) + val)
@@ -2599,3 +2714,3 @@
-	ele(2).SetValue ((val \ 10000) mod 10)*2 + ofs
-	ele(3).SetValue ((val \ 1000) mod 10)*2 + ofs
-	ele(4).SetValue ele(5)*2 + ofs
+	ele(2).SetValue 2 * ((val \ 10000) mod 10) + ofs
+	ele(3).SetValue 2 * ((val \ 1000) mod 10) + ofs
+	ele(4).SetValue 2 * ele(5) + ofs
@@ -2619 +2734 @@
-			ele(2).SetValue (ele(3) + val)*2 + ofs
+			ele(2).SetValue 2 * (ele(3) + val) + ofs
@@ -2621 +2736 @@
-			ele(2).SetValue OpMenuTmp(OpMenuLine)*2 + ofs
+			ele(2).SetValue 2 * OpMenuTmp(OpMenuLine) + ofs
@@ -2625 +2740 @@
-			ele(1).SetValue ele(2)*2 + ofs
+			ele(1).SetValue 2 * ele(2) + ofs
@@ -2676 +2791 @@
-			ele(2).SetValue (ele(3) + val)*2 + 1
+			ele(2).SetValue 2 * (ele(3) + val) + 1
@@ -2682 +2797 @@
-			ele(2).SetValue val*2 + 1
+			ele(2).SetValue 2 * val + 1
@@ -2713 +2828 @@
-							o(2).SetValue (o(3) + val)*2
+							o(2).SetValue 2 * (o(3) + val)
@@ -2716 +2831 @@
-							o(2).SetValue val*2
+							o(2).SetValue 2 * val
