--- Nine Ball (Stern 1980).vbs.original
+++ Nine Ball (Stern 1980).vbs
@@ -1591,6 +1591,39 @@
 '		DROP TARGETS INITIALIZATION
 '******************************************************
 
+Class DropTarget
+  Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+  Public Property Let Secondary(input): Set m_secondary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+  Public Property Let IsDropped(input): m_isDropped = input: End Property
+
+  Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+    Set m_primary = primary
+    Set m_secondary = secondary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+    m_isDropped = isDropped
+
+    Set Init = Me
+  End Function
+End Class
+
 'Define a variable for each drop target
 Dim DT9, DT10, DT11, DT12, DT13, DT14, DT15, DT16, DT18, DT19, DT20, DT21, DT22, DT23, DT24
 
@@ -1611,27 +1644,27 @@
 '	Values for annimate: 1 - bend target (hit to primary), 2 - drop target (hit to secondary), 3 - brick target (high velocity hit to secondary), -1 - raise target 
 
 ' 1 Bank
-DT21 = Array(sw21, sw21y, psw21, 21, 0)
+Set DT21 = (new DropTarget)(sw21, sw21y, psw21, 21, 0, false)
 
 ' 2 Bank
-DT18 = Array(sw18, sw18y, psw18, 18, 0)
-DT19 = Array(sw19, sw19y, psw19, 19, 0)
-DT20 = Array(sw20, sw20y, psw20, 20, 0)
+Set DT18 = (new DropTarget)(sw18, sw18y, psw18, 18, 0, false)
+Set DT19 = (new DropTarget)(sw19, sw19y, psw19, 19, 0, false)
+Set DT20 = (new DropTarget)(sw20, sw20y, psw20, 20, 0, false)
 
 ' 3 Bank
-DT22 = Array(sw22, sw22y, psw22, 22, 0)
-DT23 = Array(sw23, sw23y, psw23, 23, 0)
-DT24 = Array(sw24, sw24y, psw24, 24, 0)
+Set DT22 = (new DropTarget)(sw22, sw22y, psw22, 22, 0, false)
+Set DT23 = (new DropTarget)(sw23, sw23y, psw23, 23, 0, false)
+Set DT24 = (new DropTarget)(sw24, sw24y, psw24, 24, 0, false)
 
 ' Meteor Bank
-DT9 = Array(sw9, sw9y, psw9, 9, 0)
-DT10 = Array(sw10, sw10y, psw10, 10, 0)
-DT11 = Array(sw11, sw11y, psw11, 11, 0)
-DT12 = Array(sw12, sw12y, psw12, 12, 0)
-DT13 = Array(sw13, sw13y, psw13, 13, 0)
-DT14 = Array(sw14, sw14y, psw14, 14, 0)
-DT15 = Array(sw15, sw15y, psw15, 15, 0)
-DT16 = Array(sw16, sw16y, psw16, 16, 0)
+Set DT9 = (new DropTarget)(sw9, sw9y, psw9, 9, 0, false)
+Set DT10 = (new DropTarget)(sw10, sw10y, psw10, 10, 0, false)
+Set DT11 = (new DropTarget)(sw11, sw11y, psw11, 11, 0, false)
+Set DT12 = (new DropTarget)(sw12, sw12y, psw12, 12, 0, false)
+Set DT13 = (new DropTarget)(sw13, sw13y, psw13, 13, 0, false)
+Set DT14 = (new DropTarget)(sw14, sw14y, psw14, 14, 0, false)
+Set DT15 = (new DropTarget)(sw15, sw15y, psw15, 15, 0, false)
+Set DT16 = (new DropTarget)(sw16, sw16y, psw16, 16, 0, false)
 
 'Add all the Drop Target Arrays to Drop Target Animation Array
 ' DTAnimationArray = Array(DT1, DT2, ....)
@@ -1664,10 +1697,10 @@
 	i = DTArrayID(switch)
 
 	PlayTargetSound
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
 	End If
 
 	DoDTAnim
@@ -1677,7 +1710,7 @@
 	Dim i
 	i = DTArrayID(switch)
 
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
 	DoDTAnim
 End Sub
 
@@ -1685,14 +1718,14 @@
 	Dim i
 	i = DTArrayID(switch)
 
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
 	DoDTAnim
 End Sub
 
 Function DTArrayID(switch)
 	Dim i
 	For i = 0 to uBound(DTArray) 
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
 	Next
 End Function
 
@@ -1747,7 +1780,7 @@
 Sub DoDTAnim()
 	Dim i
 	For i=0 to Ubound(DTArray)
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
 	Next
 	'table specific code
 '	DTShadow psw25, drop4shadow, "drop4"
