--- Dream Daddy1.5.vbs.original
+++ Dream Daddy1.5.vbs
@@ -72 +72 @@
-Const typefont = "Bebas Neue Regular"
+Const typefont = "BebasNeue-Regular"
@@ -98,0 +99 @@
+Sub InitPuP()
@@ -133,0 +135 @@
+End Sub
@@ -202,0 +205,2 @@
+    InitPuP
+    SetupPuP
@@ -1879 +1883 @@
-				PuPlayer.playlistplayex pBackglass,"BG","Backglass2.mp4",0,1
+				PuPlayer.playlistplayex pBackglass,"Backglass","Backglass2.mp4",0,1
@@ -1890 +1894 @@
-	PuPlayer.playlistplayex pBackglass,"BG","Backglass3.mp4",0,1
+	PuPlayer.playlistplayex pBackglass,"Backglass","Backglass3.mp4",0,1
@@ -2190,0 +2195,2 @@
+
+Sub SetupPuP()
@@ -2299,0 +2306 @@
+End Sub
