--- Cybernaut darkmod.vbs.original
+++ Cybernaut darkmod.vbs
@@ -46,3 +46,11 @@
-
-
-
+Const sSaucer=9'1
+Const sLT=10'2              drop target reset
+Const sBumper=11'3         
+Const sRightSlingshot=12'4
+Const sLeftSlingshot=13'5
+Const sBallRelease=14'6     outhole
+Const sKnocker=15'7
+Const sGate=17'8            left gate
+Const sCLo=18'9             ?
+Const sEnable=19'10         K1 reley ( flipper enable )
+Const sRightGate=20'11
@@ -84,13 +91,0 @@
-
-
-Const sSaucer=9'1
-Const sLT=10'2              drop target reset
-Const sBumper=11'3         
-Const sRightSlingshot=12'4
-Const sLeftSlingshot=13'5
-Const sBallRelease=14'6     outhole
-Const sKnocker=15'7
-Const sGate=17'8            left gate
-Const sCLo=18'9             ?
-Const sEnable=19'10         K1 reley ( flipper enable )
-Const sRightGate=20'11
