--- Cybernaut.vbs.original
+++ Cybernaut.vbs
@@ -55,6 +55,18 @@
 Primitive13.visible=0
 End if
 
+Const sSaucer=9'1
+Const sLT=10'2              drop target reset
+Const sBumper=11'3         
+Const sRightSlingshot=12'4
+Const sLeftSlingshot=13'5
+Const sBallRelease=14'6     outhole
+Const sKnocker=15'7
+Const sGate=17'8            left gate
+Const sCLo=18'9             ?
+Const sEnable=19'10         K1 reley ( flipper enable )
+Const sRightGate=20'11
+
 '*************************************************************
 'Solenoid Call backs
 '**********************************************************************************************************
@@ -90,19 +102,6 @@
 End Sub
 
 
-Const sSaucer=9'1
-Const sLT=10'2              drop target reset
-Const sBumper=11'3         
-Const sRightSlingshot=12'4
-Const sLeftSlingshot=13'5
-Const sBallRelease=14'6     outhole
-Const sKnocker=15'7
-Const sGate=17'8            left gate
-Const sCLo=18'9             ?
-Const sEnable=19'10         K1 reley ( flipper enable )
-Const sRightGate=20'11
-
-
 '**********************************************************************************************************
 
  Sub FlipperTimer_Timer
