--- Space Shuttle (Taito do Brasil - 1982) v4.vbs.original
+++ Space Shuttle (Taito do Brasil - 1982) v4.vbs
@@ -962,9 +962,7 @@
 Dim NVramPatchCoinCnt
 
 Function GetNVramPath()
-    Dim WshShell
-    Set WshShell = CreateObject("WScript.Shell")
-    GetNVramPath = WshShell.RegRead("HKCU\Software\Freeware\Visual PinMame\globals\nvram_directory")
+    GetNVramPath = "pinmame\nvram"
 End function
 
 Function FileExists(FileName)
