--- AC-DC_Luci-1.5.vbs.original	2024-01-16 16:16:27.783069511 +0100
+++ AC-DC_Luci-1.5.vbs	2024-01-16 16:14:02.320605853 +0100
@@ -98,6 +98,8 @@
 Const Ballsize = 51
 Const BallMass = 1.1
 
+Const cGameName = "acd_170h"
+
 Const UseVPMModSol = True
 Dim UseVPMDMD,CustomDMD,DesktopMode
 DesktopMode = ACDC.ShowDT : CustomDMD = False
@@ -127,8 +129,6 @@
 '						 INIT TABLE
 '************************************************************************
 
-Const cGameName = "acd_170h"
-
 Dim bsTrough, bsTEject, bsBellEject, bsLowEject, PlungerIM, ACDCBank, TNTBank
 
 Sub ACDC_Init
