--- The Beatles_007.vbs.original
+++ The Beatles_007.vbs
@@ -199 +198,0 @@
-	resetbackglass
@@ -222 +221,2 @@
- 
+	PupInit
+	resetbackglass
@@ -1744,0 +1745,14 @@
+Sub chilloutthemusic
+	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":10 }"
+	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":10 }"
+	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 9, ""FN"":11, ""VL"":10 }"
+	vpmtimer.addtimer 2200, "turnitbackup '"
+End Sub
+
+Sub turnitbackup
+	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":99 }"
+	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":99 }"
+	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 9, ""FN"":11, ""VL"":99 }"
+End Sub
+
+Sub PupInit
@@ -1781,13 +1794,0 @@
-Sub chilloutthemusic
-	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":10 }"
-	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":10 }"
-	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 9, ""FN"":11, ""VL"":10 }"
-	vpmtimer.addtimer 2200, "turnitbackup '"
-End Sub
-
-Sub turnitbackup
-	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":99 }"
-	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":99 }"
-	PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 9, ""FN"":11, ""VL"":99 }"
-End Sub
-
@@ -1869 +1870 @@
-
+End Sub
