--- "./Guns N Roses (Data East 1994) VPW v1.1.vbs.original"	2023-12-04 23:01:12.396841723 +0100
+++ "./Guns N Roses (Data East 1994) VPW v1.1.vbs"	2023-12-04 23:06:21.047188157 +0100
@@ -56,7 +56,7 @@
 '*************************************************************
 ' VR Room Auto-Detect
 '*************************************************************
-Dim VR_Obj, UseVPMDMD
+Dim VR_Obj, UseVPMColoredDMD
 
 Dim DesktopMode: DesktopMode = Table1.ShowDT
 
@@ -65,17 +65,19 @@
 	For Each VR_Obj in VRCabinet : VR_Obj.Visible = 1 : Next
 	For Each VR_Obj in VRRoom : VR_Obj.Visible = 1 : Next
 	For each VR_Obj in VRBackglass : VR_Obj.visible = 1 : Next
-	UseVPMDMD = True
+	UseVPMColoredDMD = True
 	Setbackglass
 Else
 	'Pincab_Bottom.Visible = 0
 	For Each VR_Obj in VRCabinet : VR_Obj.Visible = 0 : Next
 	For Each VR_Obj in VRRoom : VR_Obj.Visible = 0 : Next
 	For each VR_Obj in VRBackglass : VR_Obj.visible = 0 : Next
-	UseVPMDMD = DesktopMode
+	UseVPMColoredDMD = DesktopMode
 	Pincab_Rails.visible = DesktopMode
 End If
 
+ScoreText.intensityScale=2 'boosts luminosity
+
 LoadVPM "01120100", "de.vbs", 3.02
 
 '******************************************************
