--- /Users/francisdb/vpinball/tables/Seawitch (Stern 1980).vbs.original	2023-06-15 11:40:52
+++ /Users/francisdb/vpinball/tables/Seawitch (Stern 1980).vbs	2023-06-15 11:40:37
@@ -1277,7 +1277,40 @@
 '******************************************************
 '		DROP TARGETS INITIALIZATION
 '******************************************************
+
+Class DropTarget
+	Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+  
+	Public Property Get Primary(): Set Primary = m_primary: End Property
+	Public Property Let Primary(input): Set m_primary = input: End Property
+  
+	Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+	Public Property Let Secondary(input): Set m_secondary = input: End Property
+  
+	Public Property Get Prim(): Set Prim = m_prim: End Property
+	Public Property Let Prim(input): Set m_prim = input: End Property
+  
+	Public Property Get Sw(): Sw = m_sw: End Property
+	Public Property Let Sw(input): m_sw = input: End Property
+  
+	Public Property Get Animate(): Animate = m_animate: End Property
+	Public Property Let Animate(input): m_animate = input: End Property
 
+	Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+	Public Property Let IsDropped(input): m_isDropped = input: End Property
+  
+	Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+	  Set m_primary = primary
+	  Set m_secondary = secondary
+	  Set m_prim = prim
+	  m_sw = sw
+	  m_animate = animate
+	  m_isDropped = isDropped
+  
+	  Set Init = Me
+	End Function
+  End Class
+
 'Define a variable for each drop target
 Dim DT21, DT22, DT23, DT24, DT29, DT30, DT31, DT32, DT38, DT39, DT40
 
\ No newline at end of file
@@ -1298,21 +1331,21 @@
 '	Values for annimate: 1 - bend target (hit to primary), 2 - drop target (hit to secondary), 3 - brick target (high velocity hit to secondary), -1 - raise target 
 
 ' 1 Bank
-DT21 = Array(sw21, sw21y, psw21, 21, 0)
-DT22 = Array(sw22, sw22y, psw22, 22, 0)
-DT23 = Array(sw23, sw23y, psw23, 23, 0)
-DT24 = Array(sw24, sw24y, psw24, 24, 0)
+Set DT21 = (new DropTarget)(sw21, sw21y, psw21, 21, 0, false)
+Set DT22 = (new DropTarget)(sw22, sw22y, psw22, 22, 0, false)
+Set DT23 = (new DropTarget)(sw23, sw23y, psw23, 23, 0, false)
+Set DT24 = (new DropTarget)(sw24, sw24y, psw24, 24, 0, false)
 
 ' 2 Bank
-DT29 = Array(sw29, sw29y, psw29, 29, 0)
-DT30 = Array(sw30, sw30y, psw30, 30, 0)
-DT31 = Array(sw31, sw31y, psw31, 31, 0)
-DT32 = Array(sw32, sw32y, psw32, 32, 0)
+Set DT29 = (new DropTarget)(sw29, sw29y, psw29, 29, 0, false)
+Set DT30 = (new DropTarget)(sw30, sw30y, psw30, 30, 0, false)
+Set DT31 = (new DropTarget)(sw31, sw31y, psw31, 31, 0, false)
+Set DT32 = (new DropTarget)(sw32, sw32y, psw32, 32, 0, false)
 
 ' 3 Bank
-DT38 = Array(sw38, sw38y, psw38, 38, 0)
-DT39 = Array(sw39, sw39y, psw39, 39, 0)
-DT40 = Array(sw40, sw40y, psw40, 40, 0)
+Set DT38 = (new DropTarget)(sw38, sw38y, psw38, 38, 0, false)
+Set DT39 = (new DropTarget)(sw39, sw39y, psw39, 39, 0, false)
+Set DT40 = (new DropTarget)(sw40, sw40y, psw40, 40, 0, false)
 
 'Add all the Drop Target Arrays to Drop Target Animation Array
 ' DTAnimationArray = Array(DT1, DT2, ....)
\ No newline at end of file
@@ -1345,9 +1378,9 @@
 	i = DTArrayID(switch)
 
 	PlayTargetSound
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
 	End If
 	DoDTAnim
 End Sub
\ No newline at end of file
@@ -1356,7 +1389,7 @@
 	Dim i
 	i = DTArrayID(switch)
 
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
 	DoDTAnim
 End Sub
 
\ No newline at end of file
@@ -1364,14 +1397,14 @@
 	Dim i
 	i = DTArrayID(switch)
 
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
 	DoDTAnim
 End Sub
 
 Function DTArrayID(switch)
 	Dim i
 	For i = 0 to uBound(DTArray) 
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
 	Next
 End Function
 
\ No newline at end of file
@@ -1433,7 +1466,7 @@
 Sub DoDTAnim()
 	Dim i
 	For i=0 to Ubound(DTArray)
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
 	Next
 
 	'table specific code
\ No newline at end of file
