--- "Three Angels (Original 2018) LW.vbs.original"	2025-03-03 18:36:27.285232231 +0100
+++ "Three Angels (Original 2018) LW.vbs"	2025-03-03 18:32:43.105787397 +0100
@@ -333,7 +333,7 @@
 
 'Note open Ultradmd and right click on window to get the various sizes in decimal 
 
-GetDMDColor
+'GetDMDColor
 Sub GetDMDColor
 Dim WshShell,filecheck,directory
 Set WshShell = CreateObject("WScript.Shell")
@@ -62828,7 +62828,7 @@
 	DMD_DisplayScene ""," . 0 0 0 .0 0 0" & DoorJackpotNumber, UltraDMD_Animation_None, UltraDMD_deOn, UltraDMD_Animation_None
 	'DMD2.QueueText DoorJackpotNumber & " . 0 0 0 .0 0 0", seBlinkMask, 1000 : 'DMDB.QueueText DoorJackpotNumber & " . 0 0 0 .0 0 0", seBlinkMask, 1000
 	
-	AddScore(1000000) * DoorJackpotNumber  'This jackpot is a little harder.
+	AddScore(1000000 * DoorJackpotNumber)  'This jackpot is a little harder.
 	bonuspoints(1)=bonuspoints(1)+100000
 
 
