--- Three Angels (Original 2018) LW.vbs.original
+++ Three Angels (Original 2018) LW.vbs
@@ -335,7 +335,7 @@
 
 'Note open Ultradmd and right click on window to get the various sizes in decimal 
 
-GetDMDColor
+'GetDMDColor
 Sub GetDMDColor
 Dim WshShell,filecheck,directory
 Set WshShell = CreateObject("WScript.Shell")
@@ -62865,7 +62865,7 @@
 	DMD_DisplayScene ""," . 0 0 0 .0 0 0" & DoorJackpotNumber, UltraDMD_Animation_None, UltraDMD_deOn, UltraDMD_Animation_None
 	'DMD2.QueueText DoorJackpotNumber & " . 0 0 0 .0 0 0", seBlinkMask, 1000 : 'DMDB.QueueText DoorJackpotNumber & " . 0 0 0 .0 0 0", seBlinkMask, 1000
 	
-	AddScore(1000000) * DoorJackpotNumber  'This jackpot is a little harder.
+	AddScore 1000000 * DoorJackpotNumber  'This jackpot is a little harder.
 	bonuspoints(1)=bonuspoints(1)+100000
 
 
