--- Four Seasons (Gottlieb 1968)_Teisen_MOD.vbs.original
+++ Four Seasons (Gottlieb 1968)_Teisen_MOD.vbs
@@ -1222,3 +1222,3 @@
-		Me(Idx).IsDropped=True
-		Me(Idx+20).IsDropped=True
-		Me(Idx+40).enabled=False
+		Spinner(Idx).IsDropped=True
+		Spinner(Idx+20).IsDropped=True
+		Spinner(Idx+40).enabled=False
@@ -1230,3 +1230,3 @@
-		Me(Idx-20).IsDropped=True
-		Me(Idx).IsDropped=True
-		Me(Idx+20).enabled=False
+		Spinner(Idx-20).IsDropped=True
+		Spinner(Idx).IsDropped=True
+		Spinner(Idx+20).enabled=False
@@ -1235,3 +1235,3 @@
-	Me(SpinPos).IsDropped=False
-	Me(SpinPos+20).IsDropped=False
-	Me(SpinPos+40).enabled=True
+	Spinner(SpinPos).IsDropped=False
+	Spinner(SpinPos+20).IsDropped=False
+	Spinner(SpinPos+40).enabled=True
@@ -1274,3 +1274,3 @@
-		Me(Idx).IsDropped=True
-		Me(Idx+20).IsDropped=True
-		Me(Idx+40).enabled=False
+		Spinner2(Idx).IsDropped=True
+		Spinner2(Idx+20).IsDropped=True
+		Spinner2(Idx+40).enabled=False
@@ -1282,3 +1282,3 @@
-		Me(Idx-20).IsDropped=True
-		Me(Idx).IsDropped=True
-		Me(Idx+20).enabled=False
+		Spinner2(Idx-20).IsDropped=True
+		Spinner2(Idx).IsDropped=True
+		Spinner2(Idx+20).enabled=False
@@ -1287,3 +1287,3 @@
-	Me(SpinPos2).IsDropped=False
-	Me(SpinPos2+20).IsDropped=False
-	Me(SpinPos2+40).enabled=True
+	Spinner2(SpinPos2).IsDropped=False
+	Spinner2(SpinPos2+20).IsDropped=False
+	Spinner2(SpinPos2+40).enabled=True
