--- monkeyisland.vbs.original
+++ monkeyisland.vbs
@@ -2444 +2444 @@
-                DMDScene.GetImage("Dig" & i).SetBounds (i - dCharsPerLine(1)) * 8, 3, 8, 16
+                DMDScene.GetImage("Dig" & i).SetBounds 8 * (i - dCharsPerLine(1)), 3, 8, 16
@@ -4395 +4395 @@
-	Kicker002.CreateSizedBall(BallSize/2).ID = MyTroughLR(0)
+	Kicker002.CreateSizedBall(BallSize/2).UserValue = MyTroughLR(0)
@@ -4416 +4416 @@
-	Kicker006.CreateSizedBall(BallSize/2).ID = MyTroughTreasure(0)
+	Kicker006.CreateSizedBall(BallSize/2).UserValue = MyTroughTreasure(0)
@@ -4433 +4433 @@
-	Kicker003.CreateSizedBall(BallSize/2).ID = MyTroughLeChuck(0)
+	Kicker003.CreateSizedBall(BallSize/2).UserValue = MyTroughLeChuck(0)
@@ -4447 +4447 @@
-	Kicker007.CreateSizedBall(BallSize/2).ID = MyTroughTreasure(0)
+	Kicker007.CreateSizedBall(BallSize/2).UserValue = MyTroughTreasure(0)
@@ -4458 +4458 @@
-	MyTroughLR(UBound(MyTroughLR)) = tBall.ID
+	MyTroughLR(UBound(MyTroughLR)) = tBall.UserValue
@@ -4465 +4465 @@
-	MyTroughTreasure(Ubound(MyTroughTreasure)) = tBall.ID
+	MyTroughTreasure(Ubound(MyTroughTreasure)) = tBall.UserValue
@@ -4471 +4471 @@
-	MyTroughLeChuck(Ubound(MyTroughLeChuck)) = tBall.ID
+	MyTroughLeChuck(Ubound(MyTroughLeChuck)) = tBall.UserValue
