--- Starship Troopers (Sega 1997) VPW Mod v2.0.vbs.original
+++ Starship Troopers (Sega 1997) VPW Mod v2.0.vbs
@@ -1327 +1327 @@
-	If FlasherOnR = False then FlasherTimer1.Enabled = 1: Else 
+	If FlasherOnR = False then FlasherTimer1.Enabled = 1
@@ -1358 +1358 @@
-	If FlasherOnG = False then FlasherTimer3.Enabled = 1: Else 
+	If FlasherOnG = False then FlasherTimer3.Enabled = 1 
@@ -1389 +1389 @@
-	If FlasherOnY = False then FlasherTimer5.Enabled = 1: Else 
+	If FlasherOnY = False then FlasherTimer5.Enabled = 1 
@@ -1419 +1419 @@
-	If FlasherOnB = False then FlasherTimer7.Enabled = 1: Else 
+	If FlasherOnB = False then FlasherTimer7.Enabled = 1 
