--- ./Bounty Hunter (Gottlieb 1985)/Bounty Hunter (Gottlieb 1985).vbs	2023-11-29 07:19:41
+++ ./Bounty Hunter (Gottlieb 1985)/Bounty Hunter (Gottlieb 1985).vbs.dmd	2023-11-29 07:19:41
@@ -19,10 +19,25 @@
 Const cGameName="bountyh",UseSolenoids=2,UseLamps=0,UseGI=0,SSolenoidOn="Solenoid",SSolenoidOff="SolOff"
 Const SCoin=""
 
-LoadVPM "01120100","sys80.vbs",3.02
+'dmd hack
+'LoadVPM "01120100","sys80.vbs",3.02
+'end of dmd hack
 
 Dim DesktopMode: DesktopMode = Table1.ShowDT
 
+'dmd hack
+Dim UseVPMDMD
+UseVPMDMD=DesktopMode
+Sub ImplicitDMD_Init
+   Me.x = 30
+   Me.y = 30
+   Me.width = 128 * 2
+   Me.height = 32 * 2
+   Me.visible = true
+End Sub
+LoadVPM "01120100","sys80.vbs",3.02
+'end of dmd hack
+
 If DesktopMode = True Then 'Show Desktop components
 displaytimer.enabled=1
 Else
