--- Three Angels (Original 2018) 1.3.vbs.original	2023-07-05 11:14:17
+++ Three Angels (Original 2018) 1.3.vbs	2023-07-05 11:14:00
@@ -320,6 +320,7 @@
 Sub GetDMDColor
 Dim WshShell,filecheck,directory
 Set WshShell = CreateObject("WScript.Shell")
+  If Not WshShell Is Nothing Then
 If DMDSize then
 WshShell.RegWrite "HKCU\Software\UltraDMD\w",DMDWidth,"REG_DWORD"
 WshShell.RegWrite "HKCU\Software\UltraDMD\h",DMDHeight,"REG_DWORD"
\ No newline at end of file
@@ -330,6 +331,7 @@
 End if
 WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
 WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
+  End If
 End Sub
 '---------------------------------------------------
 
\ No newline at end of file
@@ -62802,7 +62804,7 @@
 	DMD_DisplayScene ""," . 0 0 0 .0 0 0" & DoorJackpotNumber, UltraDMD_Animation_None, UltraDMD_deOn, UltraDMD_Animation_None
 	'DMD2.QueueText DoorJackpotNumber & " . 0 0 0 .0 0 0", seBlinkMask, 1000 : 'DMDB.QueueText DoorJackpotNumber & " . 0 0 0 .0 0 0", seBlinkMask, 1000
 	
-	AddScore(1000000) * DoorJackpotNumber  'This jackpot is a little harder.
+	AddScore(1000000 * DoorJackpotNumber)  'This jackpot is a little harder.
 	bonuspoints(1)=bonuspoints(1)+100000
 
 
\ No newline at end of file
