--- Junk Yard (Williams 1996) v1.83.vbs.original
+++ Junk Yard (Williams 1996) v1.83.vbs
@@ -887,7 +887,7 @@
 '-----------------------------------
 Dim Wrecker,Wrecker2,bottomlimit,XBallX,YBallX,ScaleFactor,LowerBottomLimit,UpperBottomLimit,Wreckerballsize
 Dim HoldCrane,WreckBallCenterX,WreckBallCenterY
-LowerBottomLimit = 30 
+LowerBottomLimit = 47
 UpperBottomLimit = 120
 bottomlimit = LowerBottomLimit
 HoldCrane = False
