--- ./X-Files Hanibal 4k LED Edition/X-Files Hanibal 4k LED Edition.vbs.original	2024-07-28 17:28:29
+++ ./X-Files Hanibal 4k LED Edition/X-Files Hanibal 4k LED Edition.vbs	2024-07-28 16:53:33
@@ -15,7 +15,16 @@
 On Error Goto 0
 
 Dim UseVPMDMD
-UseVPMDMD = 1
+UseVPMDMD = True
+Sub ImplicitDMD_Init
+   Me.x = 60
+   Me.y = 30
+   Me.width = 128 * 2
+   Me.height = 32 * 2
+   Me.visible = true
+   Me.fontColor = RGB(255, 255, 255)
+   Me.intensityScale = 1.5
+End Sub
 
 LoadVPM "01560000", "SEGA.VBS", 3.26
 
