--- Junkyard Cats_1.07.vbs.original
+++ Junkyard Cats_1.07.vbs
@@ -511,4 +511,4 @@
-        : Controller.B2SSetData 9, 0
-        : Controller.B2SSetData 10, 0
-        : Controller.B2SSetData 11, 0
-        : Controller.B2SSetData 12, 0
+        Controller.B2SSetData 9, 0
+        Controller.B2SSetData 10, 0
+        Controller.B2SSetData 11, 0
+        Controller.B2SSetData 12, 0
@@ -1150,5 +1150,5 @@
-    :Controller.B2SSetData 9, 0
-:    Controller.B2SSetData 10, 0
-    :Controller.B2SSetData 11, 0
-    :Controller.B2SSetData 12, 0
-    :Controller.B2SSetData 13, 0
+    Controller.B2SSetData 9, 0
+    Controller.B2SSetData 10, 0
+    Controller.B2SSetData 11, 0
+    Controller.B2SSetData 12, 0
+    Controller.B2SSetData 13, 0
@@ -1216,8 +1216,8 @@
-   :Controller.B2SSetData 1, 0
-   :Controller.B2SSetData 2, 0
-   :Controller.B2SSetData 3, 0
-   :Controller.B2SSetData 4, 0
-   :Controller.B2SSetData 5, 0
-   :Controller.B2SSetData 6, 0
-   :Controller.B2SSetData 7, 0
-   :Controller.B2SSetData 8, 0
+    Controller.B2SSetData 1, 0
+    Controller.B2SSetData 2, 0
+    Controller.B2SSetData 3, 0
+    Controller.B2SSetData 4, 0
+    Controller.B2SSetData 5, 0
+    Controller.B2SSetData 6, 0
+    Controller.B2SSetData 7, 0
+    Controller.B2SSetData 8, 0
@@ -1369,5 +1369,5 @@
-: Controller.B2SSetData 9, 0
-: Controller.B2SSetData 10, 0
-: Controller.B2SSetData 11, 0
-: Controller.B2SSetData 12, 0
-: Controller.B2SSetData 13, 0
+  Controller.B2SSetData 9, 0
+  Controller.B2SSetData 10, 0
+  Controller.B2SSetData 11, 0
+  Controller.B2SSetData 12, 0
+  Controller.B2SSetData 13, 0
