--- "Spooky_Wednesday VPX 2024.vbs.original"	2024-10-10 17:17:45.844604100 +0200
+++ "Spooky_Wednesday VPX 2024.vbs"	2024-10-10 17:12:22.623750252 +0200
@@ -46,7 +46,7 @@
 
 '************ PuP-Pack Startup **************
 
-PuPStart(cPuPPack) 'Check for PuP - If found, then start Pinup Player / PuP-Pack
+' PuPStart(cPuPPack) 'Check for PuP - If found, then start Pinup Player / PuP-Pack
 
 
 
@@ -97,21 +97,21 @@
 
 'Note open Ultradmd and right click on window to get the various sizes in decimal 
 
-GetDMDColor
-Sub GetDMDColor
-Dim WshShell,filecheck,directory
-Set WshShell = CreateObject("WScript.Shell")
-If DMDSize then
-WshShell.RegWrite "HKCU\Software\UltraDMD\w",DMDWidth,"REG_DWORD"
-WshShell.RegWrite "HKCU\Software\UltraDMD\h",DMDHeight,"REG_DWORD"
-End if
-If DMDPosition then
-WshShell.RegWrite "HKCU\Software\UltraDMD\x",DMDPosX,"REG_DWORD"
-WshShell.RegWrite "HKCU\Software\UltraDMD\y",DMDPosY,"REG_DWORD"
-End if
-WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
-WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
-End Sub
+' GetDMDColor
+' Sub GetDMDColor
+' Dim WshShell,filecheck,directory
+' Set WshShell = CreateObject("WScript.Shell")
+' If DMDSize then
+' WshShell.RegWrite "HKCU\Software\UltraDMD\w",DMDWidth,"REG_DWORD"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\h",DMDHeight,"REG_DWORD"
+' End if
+' If DMDPosition then
+' WshShell.RegWrite "HKCU\Software\UltraDMD\x",DMDPosX,"REG_DWORD"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\y",DMDPosY,"REG_DWORD"
+' End if
+' WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
+' End Sub
 '---------------------------------------------------
 
 
@@ -216,6 +216,8 @@
     LoadEM
     Dim i
     Randomize
+    PuPStart(cPuPPack) 'Check for PuP - If found, then start Pinup Player / PuP-Pack
+    InitPuP
    '****************************************************************************************************************************************
    'EVERYTHING FROM LINE THIS LINE TO LINE 415 IS EVERTHING THING THAT NEEDS TO BE SHUT OFF OR TURNED ON WHEN THE TABEL STARTS UP
    '****************************************************************************************************************************************
@@ -5080,11 +5082,11 @@
 '			Const pBackglassB2S = 2			
 			Const pAudio=7
 			Const pCallouts=8
-
+Sub InitPuP
 	if HasPuP Then
-	on error resume next
-	Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay") 
-	on error goto 0
+	' on error resume next
+	' Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay") 
+	' on error goto 0
 	if not IsObject(PuPlayer) then HasPuP = False
 	end If
 
@@ -5118,23 +5120,6 @@
 	End If
 	End If
 
-	Sub chilloutthemusic
-		If calloutlowermusicvol = 1 Then
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":40 }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":40 }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":40 }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":"&(calloutvol)&" }"
-			vpmtimer.addtimer 4500, "turnitbackup'"
-		End If
-	End Sub
-
-	Sub turnitbackup
-		If calloutlowermusicvol = 1 Then
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":"&(videovol)&" }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
-		End If
-	End Sub
 
 
 PuPlayer.playlistadd pMusic,"audioattract", 1 , 0
@@ -5188,7 +5173,26 @@
 PuPlayer.playlistadd pBackglass,"LostWizardMode", 1 , 2
 PuPlayer.playlistadd pBackglass,"SkillShot", 1 , 0
 End If
+End Sub
+
+
+Sub chilloutthemusic
+If calloutlowermusicvol = 1 Then
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":40 }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":40 }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":40 }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":"&(calloutvol)&" }"
+    vpmtimer.addtimer 4500, "turnitbackup'"
+End If
+End Sub
 
+Sub turnitbackup
+If calloutlowermusicvol = 1 Then
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":"&(videovol)&" }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
+End If
+End Sub
 
 '*****************************************************
 '		 THINGS MAGNET AND BALL SAVE KICKER
