--- Fog, The (Gottlieb 1979) v2.5 for 10.7.vbs.original
+++ Fog, The (Gottlieb 1979) v2.5 for 10.7.vbs
@@ -3382,0 +3383,33 @@
+Class DropTarget
+  Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+  Public Property Let Secondary(input): Set m_secondary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+  Public Property Let IsDropped(input): m_isDropped = input: End Property
+
+  Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+    Set m_primary = primary
+    Set m_secondary = secondary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+    m_isDropped = isDropped
+
+    Set Init = Me
+  End Function
+End Class
+
@@ -3402,0 +3436 @@
+'       isDropped:  Boolean which determines whether a drop target is dropped. Set to false if they are initially raised, true if initially dropped.
@@ -3405,18 +3439,18 @@
-DTR1= Array(DTRed1, DTRed1a, DTRed1p, 0, 0)
-DTR2= Array(DTRed2, DTRed2a, DTRed2p, 1, 0)
-DTR3= Array(DTRed3, DTRed3a, DTRed3p, 2, 0)
-DTR4= Array(DTRed4, DTRed4a, DTRed4p, 3, 0)
-
-DTBk1= Array(DTBlack1, DTBlack1a, DTBlack1p, 4, 0)
-DTBk2= Array(DTBlack2, DTBlack2a, DTBlack2p, 5, 0)
-DTBk3= Array(DTBlack3, DTBlack3a, DTBlack3p, 6, 0)
-DTBk4= Array(DTBlack4, DTBlack4a, DTBlack4p, 7, 0)
-DTBk5= Array(DTBlack5, DTBlack5a, DTBlack5p, 8, 0)
-DTBk6= Array(DTBlack6, DTBlack6a, DTBlack6p, 9, 0)
-DTBk7= Array(DTBlack7, DTBlack7a, DTBlack7p, 10, 0)
-DTBk8= Array(DTBlack8, DTBlack8a, DTBlack8p, 11, 0)
-
-DTBu1= Array(DTBlue1, DTBlue1a, DTBlue1p, 12, 0)
-DTBu2= Array(DTBlue2, DTBlue2a, DTBlue2p, 13, 0)
-DTBu3= Array(DTBlue3, DTBlue3a, DTBlue3p, 14, 0)
-DTBu4= Array(DTBlue4, DTBlue4a, DTBlue4p, 15, 0)
+Set DTR1= (new DropTarget)(DTRed1, DTRed1a, DTRed1p, 0, 0, false)
+Set DTR2= (new DropTarget)(DTRed2, DTRed2a, DTRed2p, 1, 0, false)
+Set DTR3= (new DropTarget)(DTRed3, DTRed3a, DTRed3p, 2, 0, false)
+Set DTR4= (new DropTarget)(DTRed4, DTRed4a, DTRed4p, 3, 0, false)
+
+Set DTBk1= (new DropTarget)(DTBlack1, DTBlack1a, DTBlack1p, 4, 0, false)
+Set DTBk2= (new DropTarget)(DTBlack2, DTBlack2a, DTBlack2p, 5, 0, false)
+Set DTBk3= (new DropTarget)(DTBlack3, DTBlack3a, DTBlack3p, 6, 0, false)
+Set DTBk4= (new DropTarget)(DTBlack4, DTBlack4a, DTBlack4p, 7, 0, false)
+Set DTBk5= (new DropTarget)(DTBlack5, DTBlack5a, DTBlack5p, 8, 0, false)
+Set DTBk6= (new DropTarget)(DTBlack6, DTBlack6a, DTBlack6p, 9, 0, false)
+Set DTBk7= (new DropTarget)(DTBlack7, DTBlack7a, DTBlack7p, 10, 0, false)
+Set DTBk8= (new DropTarget)(DTBlack8, DTBlack8a, DTBlack8p, 11, 0, false)
+
+Set DTBu1= (new DropTarget)(DTBlue1, DTBlue1a, DTBlue1p, 12, 0, false)
+Set DTBu2= (new DropTarget)(DTBlue2, DTBlue2a, DTBlue2p, 13, 0, false)
+Set DTBu3= (new DropTarget)(DTBlue3, DTBlue3a, DTBlue3p, 14, 0, false)
+Set DTBu4= (new DropTarget)(DTBlue4, DTBlue4a, DTBlue4p, 15, 0, false)
@@ -3453,3 +3487,3 @@
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
@@ -3464 +3498 @@
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
@@ -3472 +3506 @@
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
@@ -3479 +3513 @@
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
@@ -3534 +3568 @@
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
