--- Bob Marley Mod.vbs.original
+++ Bob Marley Mod.vbs
@@ -62 +62 @@
-NVramPatchLoad
+'NVramPatchLoad
@@ -156 +156,4 @@
-Sub table1_exit:NVramPatchExit:Controller.stop:End Sub
+Sub table1_exit
+    'NVramPatchExit
+    Controller.stop
+End Sub
@@ -167,4 +170,4 @@
-    myMusicFolder = "Bob Marley" 
-    Set FileSystemObject = CreateObject("Scripting.FileSystemObject")
-    musicPath = FileSystemObject.GetAbsolutePathName(".") ' get path to Visual Pinball\table
-    musicPath = Left(musicPath, Len(musicPath) - 6) + "music\" 'get path to Visual Pinball\music
+    ' myMusicFolder = "Bob Marley" 
+    ' Set FileSystemObject = CreateObject("Scripting.FileSystemObject")
+    ' musicPath = FileSystemObject.GetAbsolutePathName(".") ' get path to Visual Pinball\table
+    ' musicPath = Left(musicPath, Len(musicPath) - 6) + "music\" 'get path to Visual Pinball\music
@@ -172 +175 @@
-	Set folder = FileSystemObject.GetFolder(musicPath + myMusicFolder) 'Comment out if using custom path
+	' Set folder = FileSystemObject.GetFolder(musicPath + myMusicFolder) 'Comment out if using custom path
@@ -181,0 +185,13 @@
+    Dim music(9)
+    music(0) = "Music/Bob Marley/Could You Be Loved - Bob Marley.mp3"
+    music(1) = "Music/Bob Marley/I Know - Bob Marley.mp3"
+    music(2) = "Music/Bob Marley/Is This Love - Bob Marley.mp3"
+    music(3) = "Music/Bob Marley/No Woman No Cry - Bob Marley.mp3"
+    music(4) = "Music/Bob Marley/One Love - Bob Marley.mp3"
+    music(5) = "Music/Bob Marley/Stiff Necked Fools - Bob Marley.mp3"
+    music(6) = "Music/Bob Marley/Stir It Up - Bob Marley.mp3"
+    music(7) = "Music/Bob Marley/Sun Is Shining - Bob Marley.mp3"
+    music(8) = "Music/Bob Marley/Waiting In Vain - Bob Marley.mp3"
+    music(9) = "Music/Bob Marley/Zion Train Lyrics - Bob Marley.mp3"
+
+
@@ -183 +199 @@
-    r = INT(folder.Files.Count * Rnd + 1)
+    r = INT(UBound(music) * Rnd + 1)
@@ -185 +201 @@
-    For Each file in folder.Files
+    For Each file in music
@@ -188 +204 @@
-               PlayMusic Mid(file, Len(musicPath) + 1, 1000) 
+               PlayMusic file
@@ -604 +620 @@
-    NVramPatchKeyCheck
+    'NVramPatchKeyCheck
