--- speakeasy4.vbs.original	2023-12-12 00:48:58.640166944 +0100
+++ speakeasy4.vbs	2023-12-12 00:47:59.542702268 +0100
@@ -1775,7 +1775,7 @@
 			For each ii in Rswitches:Controller.switch(ii)=0:next
 			roulettesw.enabled = false
 			For ii = 0 to 11
-				GWHoles(ii).kick (ii+1)*30+60, (rnd(4))+12
+				GWHoles(ii).kick 30*(ii+1)+60, (rnd(4))+12
 				GWHoles(ii).enabled = false
 			Next			
 			GWRamp.collidable = 1
