--- KISS (Stern 2015)_Bigus(MOD)1.1.vbs.original
+++ KISS (Stern 2015)_Bigus(MOD)1.1.vbs
@@ -3645,8 +3645,8 @@
-    Dim WshShell,filecheck,directory
-    Set WshShell = CreateObject("WScript.Shell")
-    If DMDPosition then
-        WshShell.RegWrite "HKCU\Software\UltraDMD\x",DMDPosX,"REG_DWORD"
-        WshShell.RegWrite "HKCU\Software\UltraDMD\y",DMDPosY,"REG_DWORD"
-    End if
-    WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
-    WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
+    'Dim WshShell,filecheck,directory
+    'Set WshShell = CreateObject("WScript.Shell")
+    'If DMDPosition then
+    '    WshShell.RegWrite "HKCU\Software\UltraDMD\x",DMDPosX,"REG_DWORD"
+    '    WshShell.RegWrite "HKCU\Software\UltraDMD\y",DMDPosY,"REG_DWORD"
+    'End if
+    'WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
+    'WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
