--- Cybernaut (Bally 1985)_Bigus(MOD)1.0.vbs.original
+++ Cybernaut (Bally 1985)_Bigus(MOD)1.0.vbs
@@ -40,6 +40,19 @@
 
 Const cGameName="cybrnaut",UseSolenoids=2,UseLamps=0,UseGI=0,SSolenoidOn="SolOn",SSolenoidOff="SolOff", SCoin="coin"
 
+
+Const sSaucer=9'1
+Const sLT=10'2              drop target reset
+Const sBumper=11'3         
+Const sRightSlingshot=12'4
+Const sLeftSlingshot=13'5
+Const sBallRelease=14'6     outhole
+Const sKnocker=15'7
+Const sGate=17'8            left gate
+Const sCLo=18'9             ?
+Const sEnable=19'10         K1 reley ( flipper enable )
+Const sRightGate=20'11
+
 LoadVPM "01130100", "Bally.VBS", 3.21
 Dim DesktopMode: DesktopMode = Table1.ShowDT
 
@@ -90,19 +103,6 @@
 End Sub
 
 
-Const sSaucer=9'1
-Const sLT=10'2              drop target reset
-Const sBumper=11'3         
-Const sRightSlingshot=12'4
-Const sLeftSlingshot=13'5
-Const sBallRelease=14'6     outhole
-Const sKnocker=15'7
-Const sGate=17'8            left gate
-Const sCLo=18'9             ?
-Const sEnable=19'10         K1 reley ( flipper enable )
-Const sRightGate=20'11
-
-
 '**********************************************************************************************************
 
  Sub FlipperTimer_Timer
