--- Cactus Canyon (Bally 1998) VPW 1.1.vbs.original
+++ Cactus Canyon (Bally 1998) VPW 1.1.vbs
@@ -110,0 +111 @@
+Const tnob = 4 ' total number of balls
@@ -1928 +1928,0 @@
-Const tnob = 4 ' total number of balls
@@ -3836,0 +3837,34 @@
+Class DropTarget
+  Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+  Public Property Let Secondary(input): Set m_secondary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+  Public Property Let IsDropped(input): m_isDropped = input: End Property
+
+  Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+    Set m_primary = primary
+    Set m_secondary = secondary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+    m_isDropped = isDropped
+
+    Set Init = Me
+  End Function
+End Class
+
+
@@ -3854,0 +3889 @@
+'       isDropped:  Boolean which determines whether a drop target is dropped. Set to false if they are initially raised, true if initially dropped.
@@ -3856,4 +3891,4 @@
-DT61 = Array(sw61, sw61a, sw61p, 61, 0)
-DT62 = Array(sw62, sw62a, sw62p, 62, 0)
-DT63 = Array(sw63, sw63a, sw63p, 63, 0)
-DT64 = Array(sw64, sw64a, sw64p, 64, 0)
+Set DT61 = (new DropTarget)(sw61, sw61a, sw61p, 61, 0, false)
+Set DT62 = (new DropTarget)(sw62, sw62a, sw62p, 62, 0, false)
+Set DT63 = (new DropTarget)(sw63, sw63a, sw63p, 63, 0, false)
+Set DT64 = (new DropTarget)(sw64, sw64a, sw64p, 64, 0, false)
@@ -3890,3 +3925,3 @@
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
@@ -3901 +3936 @@
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
@@ -3909 +3944 @@
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
@@ -3916 +3951 @@
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
@@ -3978 +4013 @@
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
