--- Corinthian Master 1.3.vbs.original
+++ Corinthian Master 1.3.vbs
@@ -181,7 +181,7 @@
 			BIP = BIP + 1
 			If B2SOn Then Controller.B2SSetScore 1, PlayerScore
 			BallReleaseTimer.enabled = False
-			BallRelease.CreateSizedBall(BallSize).ID = BIP
+			BallRelease.CreateSizedBall(BallSize).UserValue = BIP
 			BallRelease.Kick 0, 7
 			DOF 122, DOFPulse
 		Else
@@ -329,20 +329,20 @@
 Sub Addpoints(ScorePar)
 		if GameStarted Then			
 						
-			If ActiveBall.ID = BIP Then CurrentBall = True Else CurrentBall = False End If 
+			If ActiveBall.UserValue = BIP Then CurrentBall = True Else CurrentBall = False End If 
 		end If
 		If ScorePar > 1 Then
 			PlayerScore = PlayerScore + ScorePar
-			debug.print "Ball " & ActiveBall.ID & " hit       " & ScorePar & " =" & PlayerScore
+			debug.print "Ball " & ActiveBall.UserValue & " hit       " & ScorePar & " =" & PlayerScore
 			If CurrentBall = True Then NextBall:End If
 		ElseIf ScorePar = 0 Then
-			debug.print "Ball " & ActiveBall.ID & " hit end   " & ScorePar & " =" & PlayerScore & " - " & TriggerEndofPlayfield.BallCntOver & " lost"
+			debug.print "Ball " & ActiveBall.UserValue & " hit end   " & ScorePar & " =" & PlayerScore & " - " & TriggerEndofPlayfield.BallCntOver & " lost"
 			If CurrentBall = True Then NextBall:End If
 		Elseif ScorePar = -1 Then
-			debug.print "Ball " & ActiveBall.ID & " unhit end " & ScorePar & " =" & PlayerScore
+			debug.print "Ball " & ActiveBall.UserValue & " unhit end " & ScorePar & " =" & PlayerScore
 		Else
 			PlayerScore = PlayerScore + ScorePar
-			debug.print "Ball " & ActiveBall.ID & " hit       " & ScorePar & " =" & PlayerScore
+			debug.print "Ball " & ActiveBall.UserValue & " hit       " & ScorePar & " =" & PlayerScore
 		End If
 		
 End Sub
