--- "Monaco (Segasa 1977)V1.4.vbs.original"	2024-01-26 15:26:43.903021348 +0100
+++ "Monaco (Segasa 1977)V1.4.vbs"	2024-01-26 15:25:24.656853705 +0100
@@ -1095,7 +1095,7 @@
 End Sub
 
 Sub Display_Match()
-	If (Match \ 10) < 5 Then  MatchReel1.SetValue(Match \ 10) + 1  Else  MatchReel2.SetValue(Match \ 10) + 1
+	If (Match \ 10) < 5 Then  MatchReel1.SetValue 1 + (Match \ 10)  Else  MatchReel2.SetValue 1 + (Match \ 10)
     If B2SOn then
         If Match = 0 then
             Controller.B2SSetMatch 100
