--- "Indiana Jones (Stern 2008)-Hanibal-2.6.vbs.original"	2024-03-10 09:51:40.247592670 +0100
+++ "Indiana Jones (Stern 2008)-Hanibal-2.6.vbs"	2024-03-10 09:48:25.906204146 +0100
@@ -2117,7 +2117,7 @@
 				If isObject(mExitKicker) Then
 					If kForce < 1 Then kForce = 1
 					kDir = kBaseDir + (Rnd - 0.5)*KickAngleVar
-					vpmTimer.AddTimer (ii-1)*200, "vpmCreateBall(" & mExitKicker.Name & ").Kick " &_
+					vpmTimer.AddTimer 200*(ii-1), "vpmCreateBall(" & mExitKicker.Name & ").Kick " &_
 					  CInt(kDir) & "," & Replace(kForce,",",".") & "," & Replace(KickZ,",",".") & " '"
 				End If
 				kForce = kForce * 0.8
