--- Sonic The Hedgehog (Brendan Bailey 2005) VPX MOD 1.33.vbs.original
+++ Sonic The Hedgehog (Brendan Bailey 2005) VPX MOD 1.33.vbs
@@ -159,4 +159,4 @@
-			Dim WshShell
-			Set WshShell = CreateObject("WScript.Shell")
-			WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor","True","REG_SZ"
-			WshShell.RegWrite "HKCU\Software\UltraDMD\color","White","REG_SZ"
+			' Dim WshShell
+			' Set WshShell = CreateObject("WScript.Shell")
+			' WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor","True","REG_SZ"
+			' WshShell.RegWrite "HKCU\Software\UltraDMD\color","White","REG_SZ"
@@ -980 +980 @@
-			DisplayB2SText (score(currentplayer)) &" " & "              BALL 1 "
+			DisplayB2SText score(currentplayer) &" " & "              BALL 1 "
@@ -984 +984 @@
-			DisplayB2SText (score(currentplayer)) &" " & "              BALL 2 "
+			DisplayB2SText score(currentplayer) &" " & "              BALL 2 "
@@ -988 +988 @@
-			DisplayB2SText (score(currentplayer)) &" " & "              BALL 3 "
+			DisplayB2SText score(currentplayer) &" " & "              BALL 3 "
