--- TimonPumbaa.vbs.original
+++ TimonPumbaa.vbs
@@ -3551 +3551 @@
-	debug.print "img=" & Flex_Overlay_image & Flex_Overlay_FrameCounter
+	'debug.print "img=" & Flex_Overlay_image & Flex_Overlay_FrameCounter
@@ -8915 +8915,2 @@
-			dy=-1*(EVAL("roachxy"&xx)(1)(roachstep)-EVAL("roachxy"&xx)(1)(roachstep-1))	'delta Y
+			Dim roachxy:roachxy=EVAL("roachxy"&xx)
+			dy=-1*(roachxy(1)(roachstep)-roachxy(1)(roachstep-1))	'delta Y
@@ -8917 +8918 @@
-			dx=EVAL("roachxy"&xx)(0)(roachstep)-EVAL("roachxy"&xx)(0)(roachstep-1)		'delta X
+			dx=roachxy(0)(roachstep)-roachxy(0)(roachstep-1)		'delta X
