--- Inhabiting Mars RC 4.vbs.original
+++ Inhabiting Mars RC 4.vbs
@@ -2311,7 +2311,7 @@
                 DMDScene.GetImage("Dig" & i).SetBounds 2 + i * 13, 3 + 32 + 2, 16, 16
             Next
             For i = 20 to 35 ' Bottom
-                DMDScene.GetImage("Dig" & i).SetBounds (i - 20) * 16, 10, 16, 16
+                DMDScene.GetImage("Dig" & i).SetBounds ((i - 20) * 16), 10, 16, 16
             Next
             FlexDMD.LockRenderThread
             FlexDMD.Stage.AddActor DMDScene
