--- Batman66_1.1.0.vbs.original	2025-02-12 21:04:20.222312499 +0100
+++ Batman66_1.1.0.vbs	2025-02-12 21:04:07.117841730 +0100
@@ -93,7 +93,7 @@
 '   PinUp Player USER Config
 '**************************
 Dim bSingleScreen:bSingleScreen=False	   ' when Playfield is in landscape and single screen this will allow PUP to work properly
-dim PuPDMDDriverType: PuPDMDDriverType=0   ' 0=LCD DMD, 2=FullDMD
+dim PuPDMDDriverType: PuPDMDDriverType=2   ' 0=LCD DMD, 2=FullDMD
 dim useHighResDMD:useHighResDMD=False	   ' if you have a high res FullDMD this will make some animtions look better
 dim useRealDMDScale : useRealDMDScale=0    ' 0 or 1 for RealDMD scaling.  Choose which one you prefer.
 dim useDMDVideos    : useDMDVideos=True	   ' true or false to use DMD splash videos.
@@ -12991,7 +12991,8 @@
 End Class 
 Dim AnimateArray(10)
 Dim aniDict
-set aniDict = New cvpmDictionary
+'set aniDict = New cvpmDictionary ' this seems to hide errors
+set aniDict = CreateObject("Scripting.Dictionary")
 Sub AnimateLabelCancel(LabelName)
 'debug.print "AnimateLabelCancel:" & LabelName
 	On error Resume next 
@@ -13015,6 +13016,10 @@
 	thisAni.ani_count=0
 	thisAni.ani_Enabled=False 
 	thisAni.ani_Cancelled=False 
+	'Add fails if the key already exists according to below linked spec.
+	'I do wonder why this does not fail on windows, but this was using a custom dictionary before?
+	'https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/add-method-dictionary#remarks
+	if aniDict.Exists(LabelName) then aniDict.Remove LabelName
 	aniDict.add LabelName, thisAni 
 	vpmTimer.AddTimer initialDelay, "aniDict.Item(""" & LabelName & """).ani_Enabled = True:tmrPupAnimation.Enabled = True '"
 End Sub
@@ -13024,7 +13029,17 @@
 	Dim Found
 	Found = False
 ''debug.print "tmrPupAnimation_Timer"
+
+	' Dim key
+	' for each key in aniDict.Keys
+	' 	debug.print "Key:" & key & " = " & TypeName(aniDict.Item(key))
+	' next
+
 	for each obj in aniDict.Items
+		'Somehow we end up with two Empty items, maybe our wine implementation of the dictionary remove is broken
+		'The more I look at this the more I get confused. Settling with a workaround for now.
+		If IsEmpty(obj)=False then
+
 		if obj.ani_Enabled and obj.ani_Cancelled=False then 
 ''debug.print "Process:" & obj.ani_LabelName & " " & obj.ani_y & " " & obj.ani_count & " " & obj.ani_endCount
 			Found=True
@@ -13039,6 +13054,8 @@
 				obj.ani_Enabled = False 
 			End if
 		End if
+
+		End If
 	Next
 
 	if Not Found Then	' Nothing active (save cycles)
