--- Batman66_1.1.0.vbs.original
+++ Batman66_1.1.0.vbs
@@ -96 +96 @@
-dim PuPDMDDriverType: PuPDMDDriverType=0   ' 0=LCD DMD, 2=FullDMD
+dim PuPDMDDriverType: PuPDMDDriverType=2   ' 0=LCD DMD, 2=FullDMD
@@ -12994 +12994,2 @@
-set aniDict = New cvpmDictionary
+'set aniDict = New cvpmDictionary ' this seems to hide errors
+set aniDict = CreateObject("Scripting.Dictionary")
@@ -13017,0 +13019,4 @@
+	'Add fails if the key already exists according to below linked spec.
+	'I do wonder why this does not fail on windows, but this was using a custom dictionary before?
+	'https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/add-method-dictionary#remarks
+	if aniDict.Exists(LabelName) then aniDict.Remove LabelName
@@ -13026,0 +13032,6 @@
+
+	' Dim key
+	' for each key in aniDict.Keys
+	' 	debug.print "Key:" & key & " = " & TypeName(aniDict.Item(key))
+	' next
+
@@ -13027,0 +13039,4 @@
+		'Somehow we end up with two Empty items, maybe our wine implementation of the dictionary remove is broken
+		'The more I look at this the more I get confused. Settling with a workaround for now.
+		If IsEmpty(obj)=False then
+
@@ -13041,0 +13057,2 @@
+
+		End If
