--- Gemini (Gottlieb 1978).vbs.original
+++ Gemini (Gottlieb 1978).vbs
@@ -1176 +1176 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1186 +1186 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1196 +1196 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1205 +1205 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1215 +1215 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1225 +1225 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1234 +1234 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1243 +1243 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
