--- "Hang Glider (Bally 1976) VPW v1.2.vbs.original"	2023-11-20 10:38:25.798265347 +0100
+++ "Hang Glider (Bally 1976) VPW v1.2.vbs"	2023-11-20 10:31:33.323515156 +0100
@@ -1420,12 +1420,45 @@
 '******************************************
 ' Drop Targets
 
+Class DropTarget
+	Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+  
+	Public Property Get Primary(): Set Primary = m_primary: End Property
+	Public Property Let Primary(input): Set m_primary = input: End Property
+  
+	Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+	Public Property Let Secondary(input): Set m_secondary = input: End Property
+  
+	Public Property Get Prim(): Set Prim = m_prim: End Property
+	Public Property Let Prim(input): Set m_prim = input: End Property
+  
+	Public Property Get Sw(): Sw = m_sw: End Property
+	Public Property Let Sw(input): m_sw = input: End Property
+  
+	Public Property Get Animate(): Animate = m_animate: End Property
+	Public Property Let Animate(input): m_animate = input: End Property
+  
+	Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+	Public Property Let IsDropped(input): m_isDropped = input: End Property
+  
+	Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+	  Set m_primary = primary
+	  Set m_secondary = secondary
+	  Set m_prim = prim
+	  m_sw = sw
+	  m_animate = animate
+	  m_isDropped = isDropped
+  
+	  Set Init = Me
+	End Function
+End Class
+
 Dim DT006, DT007, DT008, DT009, DT010
-DT006 = Array(WallTarget006, WallTarget006offset, vca_Target006, 06, 0)
-DT007 = Array(WallTarget007, WallTarget007offset, vca_Target007, 07, 0)
-DT008 = Array(WallTarget008, WallTarget008offset, vca_Target008, 08, 0)
-DT009 = Array(WallTarget009, WallTarget009offset, vca_Target009, 09, 0)
-DT010 = Array(WallTarget010, WallTarget010offset, vca_Target010, 10, 0)
+Set DT006 = (new DropTarget)(WallTarget006, WallTarget006offset, vca_Target006, 06, 0, false)
+Set DT007 = (new DropTarget)(WallTarget007, WallTarget007offset, vca_Target007, 07, 0, false)
+Set DT008 = (new DropTarget)(WallTarget008, WallTarget008offset, vca_Target008, 08, 0, false)
+Set DT009 = (new DropTarget)(WallTarget009, WallTarget009offset, vca_Target009, 09, 0, false)
+Set DT010 = (new DropTarget)(WallTarget010, WallTarget010offset, vca_Target010, 10, 0, false)
 
 Dim DTArray
 DTArray = Array(DT006, DT007, DT008, DT009, DT010)
@@ -1483,9 +1516,9 @@
         i = DTArrayID(switch)
 
 '        PlaySoundAtVol  DTHitSound, Activeball, Vol(Activeball)*22.5
-        DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-        If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-                DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+        DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+        If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+                DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
         End If
         DoDTAnim
 End Sub
@@ -1494,7 +1527,7 @@
         Dim i
         i = DTArrayID(switch)
 
-        DTArray(i)(4) = -1
+        DTArray(i).animate = -1
         DoDTAnim
 End Sub
 
@@ -1502,14 +1535,14 @@
         Dim i
         i = DTArrayID(switch)
 
-        DTArray(i)(4) = 1
+        DTArray(i).animate = 1
         DoDTAnim
 End Sub
 
 Function DTArrayID(switch)
         Dim i
         For i = 0 to uBound(DTArray) 
-                If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+                If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
         Next
 End Function
 
@@ -1577,7 +1610,7 @@
 Sub DoDTAnim()
         Dim i
         For i=0 to Ubound(DTArray)
-                DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+                DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
         Next
 End Sub
 
