--- Haunted House (Gottlieb 1982) 1.0.vbs.original
+++ Haunted House (Gottlieb 1982) 1.0.vbs
@@ -2983,2 +2983,6 @@
-Sub Sw64_Hit()Controller.Switch(64)=1:PlaySoundAt "sensor",sw64:Gate3.Damping = .85:sw64step = 0:me.timerenabled = True: End Sub
-Sub Sw64_UnHit()Controller.Switch(64)=0: End Sub
+Sub Sw64_Hit()
+	Controller.Switch(64)=1:PlaySoundAt "sensor",sw64:Gate3.Damping = .85:sw64step = 0:me.timerenabled = True
+End Sub
+Sub Sw64_UnHit()
+	Controller.Switch(64)=0
+End Sub
@@ -2986,2 +2990,6 @@
-Sub Sw35_Hit()Controller.Switch(35)=1:PlaySoundAt "sensor",sw35:Gate4.Damping = .85:sw35step = 0:me.timerenabled = True: End Sub
-Sub Sw35_UnHit()Controller.Switch(35)=0: End Sub
+Sub Sw35_Hit()
+	Controller.Switch(35)=1:PlaySoundAt "sensor",sw35:Gate4.Damping = .85:sw35step = 0:me.timerenabled = True
+End Sub
+Sub Sw35_UnHit()
+	Controller.Switch(35)=0
+End Sub
