--- Spooky Wednesday Pro.vbs.original
+++ Spooky Wednesday Pro.vbs
@@ -46,7 +46,7 @@
 
 '************ PuP-Pack Startup **************
 
-PuPStart(cPuPPack) 'Check for PuP - If found, then start Pinup Player / PuP-Pack
+' PuPStart(cPuPPack) 'Check for PuP - If found, then start Pinup Player / PuP-Pack
 
 
 
@@ -114,11 +114,11 @@
 			Const pCallouts=8
             Const pFullDMD=5        'change this number to the screen of your choosing for FullDMD
             
-
+Sub InitPuP
 	if HasPuP Then
-	on error resume next
-	Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay") 
-	on error goto 0
+	' on error resume next
+	' Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay") 
+	' on error goto 0
 	if not IsObject(PuPlayer) then HasPuP = False
 	end If
 
@@ -165,25 +165,6 @@
 	End If
 	End If  
 
-	Sub chilloutthemusic
-		If calloutlowermusicvol = 1 Then
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":40 }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":40 }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":40 }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":"&(calloutvol)&" }"
-			vpmtimer.addtimer 4500, "turnitbackup'"
-		End If
-	End Sub
-
-	Sub turnitbackup
-		If calloutlowermusicvol = 1 Then
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":"&(videovol)&" }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
-			PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
-            PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(Topper2vol)&" }"
-		End If
-	End Sub
-
 
 PuPlayer.playlistadd pMusic,"audioattract", 1 , 0
 PuPlayer.playlistadd pMusic,"audiobg", 1 , 0
@@ -238,8 +219,26 @@
 PuPlayer.playlistadd pTopper2,"Topper", 1 , 0
 PuPlayer.playlistadd pFullDMD,"fulldmd", 1 , 0
 End If
+End Sub
 
+Sub chilloutthemusic
+If calloutlowermusicvol = 1 Then
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":40 }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":40 }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":40 }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":"&(calloutvol)&" }"
+    vpmtimer.addtimer 4500, "turnitbackup'"
+End If
+End Sub
 
+Sub turnitbackup
+If calloutlowermusicvol = 1 Then
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":"&(videovol)&" }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
+    PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(Topper2vol)&" }"
+End If
+End Sub
 
 '***********************************
 'Intro art options on line 281, 2211 
@@ -266,21 +265,21 @@
 
 'Note open Ultradmd and right click on window to get the various sizes in decimal 
 
-GetDMDColor
-Sub GetDMDColor
-Dim WshShell,filecheck,directory
-Set WshShell = CreateObject("WScript.Shell")
-If DMDSize then
-WshShell.RegWrite "HKCU\Software\UltraDMD\w",DMDWidth,"REG_DWORD"
-WshShell.RegWrite "HKCU\Software\UltraDMD\h",DMDHeight,"REG_DWORD"
-End if
-If DMDPosition then
-WshShell.RegWrite "HKCU\Software\UltraDMD\x",DMDPosX,"REG_DWORD"
-WshShell.RegWrite "HKCU\Software\UltraDMD\y",DMDPosY,"REG_DWORD"
-End if
-WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
-WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
-End Sub
+' GetDMDColor
+' Sub GetDMDColor
+' Dim WshShell,filecheck,directory
+' Set WshShell = CreateObject("WScript.Shell")
+' If DMDSize then
+' WshShell.RegWrite "HKCU\Software\UltraDMD\w",DMDWidth,"REG_DWORD"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\h",DMDHeight,"REG_DWORD"
+' End if
+' If DMDPosition then
+' WshShell.RegWrite "HKCU\Software\UltraDMD\x",DMDPosX,"REG_DWORD"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\y",DMDPosY,"REG_DWORD"
+' End if
+' WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
+' End Sub
 '---------------------------------------------------
 
 
@@ -294,6 +293,8 @@
     ExecuteGlobal GetTextFile("controller.vbs")
     If Err Then MsgBox "Can't open controller.vbs"
     On Error Goto 0
+End Sub
+Sub InitTable    
     AudioAttractTimer.Enabled = True
     DiscoBallLightGlow.State=0
     FireBallDMDTrigger.Enabled = True
@@ -485,6 +486,9 @@
     LoadEM
     Dim i
     Randomize
+    PuPStart(cPuPPack) 'Check for PuP - If found, then start Pinup Player / PuP-Pack
+    InitPuP
+    InitTable
    '****************************************************************************************************************************************
    'EVERYTHING FROM LINE THIS LINE TO LINE 415 IS EVERTHING THING THAT NEEDS TO BE SHUT OFF OR TURNED ON WHEN THE TABEL STARTS UP
    '****************************************************************************************************************************************
