--- pizzatime-65.vbs.original
+++ pizzatime-65.vbs
@@ -51 +51 @@
-Const osbactive			= 1 	' Orbital Scoreboard: Set to 0 for off, 1 for only player 1 to be sent, 2 for all scores to be sent.	
+Const osbactive			= 0 	' Orbital Scoreboard: Set to 0 for off, 1 for only player 1 to be sent, 2 for all scores to be sent.	
@@ -153,0 +154,2 @@
+
+Const bHardMode	= True
@@ -1857 +1859 @@
-				AddScore (BonusMultiplier(CurrentPlayer)-1)*FormatScore(ScoreBonusAdd)  ' Subtract one since we already added score counting up total
+				AddScore FormatScore(ScoreBonusAdd)*(BonusMultiplier(CurrentPlayer)-1)  ' Subtract one since we already added score counting up total
@@ -13637,2 +13638,0 @@
-
-Const bHardMode	= True
