--- Honey (Williams 1971)V1.3.vbs.original
+++ Honey (Williams 1971)V1.3.vbs
@@ -1063,7 +1063,7 @@
 End Sub
 
 Sub Display_Match()
-	If (Match \ 10) < 4 Then  MatchReel1.SetValue(Match \ 10) + 1  Else  MatchReel2.SetValue(Match \ 10) + 1
+	If (Match \ 10) < 4 Then  MatchReel1.SetValue 1 + (Match \ 10)  Else  MatchReel2.SetValue 1 + (Match \ 10)
     If B2SOn then
         If Match = 0 then
             Controller.B2SSetMatch 100
