--- MF DOOM (GOILL773 2024) v1.1.vbs.original
+++ MF DOOM (GOILL773 2024) v1.1.vbs
@@ -91,2 +91,2 @@
-Dim re,fso,folder,files,Songs
-Songs = mp3PathArray
+'Dim re,fso,folder,files,Songs
+'Songs = mp3PathArray
@@ -93,0 +94,17 @@
+dim Songs(15)
+Songs(1) = "MFDOOM01.mp3"
+Songs(2) = "MFDOOM02.mp3"
+Songs(3) = "MFDOOM03.mp3"
+Songs(4) = "MFDOOM04.mp3"
+Songs(5) = "MFDOOM05.mp3"
+Songs(6) = "MFDOOM06.mp3"
+Songs(7) = "MFDOOM07.mp3"
+Songs(8) = "MFDOOM08.mp3"
+Songs(9) = "MFDOOM09.mp3"
+Songs(10) = "MFDOOM10.mp3"
+Songs(11) = "MFDOOM11.mp3"
+Songs(12) = "MFDOOM12.mp3"
+Songs(13) = "MFDOOM13.mp3"
+Songs(13) = "MFDOOM14.mp3"
+Songs(14) = "Attract1.mp3"
+Songs(15) = "Attract2.mp3"
