--- Space Shuttle (Williams 1984).vbs.original
+++ Space Shuttle (Williams 1984).vbs
@@ -583,0 +584,32 @@
+Class DropTarget
+	Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+  
+	Public Property Get Primary(): Set Primary = m_primary: End Property
+	Public Property Let Primary(input): Set m_primary = input: End Property
+  
+	Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+	Public Property Let Secondary(input): Set m_secondary = input: End Property
+  
+	Public Property Get Prim(): Set Prim = m_prim: End Property
+	Public Property Let Prim(input): Set m_prim = input: End Property
+  
+	Public Property Get Sw(): Sw = m_sw: End Property
+	Public Property Let Sw(input): m_sw = input: End Property
+  
+	Public Property Get Animate(): Animate = m_animate: End Property
+	Public Property Let Animate(input): m_animate = input: End Property
+  
+	Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+	Public Property Let IsDropped(input): m_isDropped = input: End Property
+  
+	Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+	  Set m_primary = primary
+	  Set m_secondary = secondary
+	  Set m_prim = prim
+	  m_sw = sw
+	  m_animate = animate
+	  m_isDropped = isDropped
+  
+	  Set Init = Me
+	End Function
+End Class
@@ -604 +636 @@
-DT20 = Array(sw20, sw20y, psw20, 20, 0)
+Set DT20 = (new DropTarget)(sw20, sw20y, psw20, 20, 0, false)
@@ -607,3 +639,3 @@
-DT33 = Array(sw33, sw33y, psw33, 33, 0)
-DT34 = Array(sw34, sw34y, psw34, 34, 0)
-DT35 = Array(sw35, sw35y, psw35, 35, 0)
+Set DT33 = (new DropTarget)(sw33, sw33y, psw33, 33, 0, false)
+Set DT34 = (new DropTarget)(sw34, sw34y, psw34, 34, 0, false)
+Set DT35 = (new DropTarget)(sw35, sw35y, psw35, 35, 0, false)
@@ -643,3 +675,3 @@
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
@@ -654 +686 @@
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
@@ -662 +694 @@
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
@@ -669 +701 @@
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
@@ -721 +753 @@
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
@@ -891,0 +924,25 @@
+Class StandupTarget
+  Private m_primary, m_prim, m_sw, m_animate
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public default Function init(primary, prim, sw, animate)
+    Set m_primary = primary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+
+    Set Init = Me
+  End Function
+End Class
+
@@ -905,7 +962,7 @@
-ST17 = Array(tsw17, primt17,17, 0)
-ST18 = Array(tsw18, primt18,18, 0)
-ST19 = Array(tsw19, primt19,19, 0)
-ST21 = Array(tsw21, primt21,21, 0)
-ST22 = Array(tsw22, primt22,22, 0)
-ST23 = Array(tsw23, primt23,23, 0)
-ST38 = Array(tsw38, primt38,38, 0)
+Set ST17 = (new StandupTarget)(tsw17, primt17,17, 0)
+Set ST18 = (new StandupTarget)(tsw18, primt18,18, 0)
+Set ST19 = (new StandupTarget)(tsw19, primt19,19, 0)
+Set ST21 = (new StandupTarget)(tsw21, primt21,21, 0)
+Set ST22 = (new StandupTarget)(tsw22, primt22,22, 0)
+Set ST23 = (new StandupTarget)(tsw23, primt23,23, 0)
+Set ST38 = (new StandupTarget)(tsw38, primt38,38, 0)
@@ -934 +991 @@
-	STArray(i)(3) =  STCheckHit(Activeball,STArray(i)(0))
+	STArray(i).animate =  STCheckHit(Activeball,STArray(i).primary)
@@ -936,2 +993,2 @@
-	If STArray(i)(3) <> 0 Then
-		DTBallPhysics Activeball, STArray(i)(0).orientation, STMass
+	If STArray(i).animate <> 0 Then
+		DTBallPhysics Activeball, STArray(i).primary.orientation, STMass
@@ -945 +1002 @@
-		If STArray(i)(2) = switch Then STArrayID = i:Exit Function 
+		If STArray(i).sw = switch Then STArrayID = i:Exit Function 
@@ -974 +1031 @@
-		STArray(i)(3) = STAnimate(STArray(i)(0),STArray(i)(1),STArray(i)(2),STArray(i)(3))
+		STArray(i).animate = STAnimate(STArray(i).primary,STArray(i).prim,STArray(i).sw,STArray(i).animate)
