--- SpaceRamp (SuperEd) v3.03b.vbs.original
+++ SpaceRamp (SuperEd) v3.03b.vbs
@@ -615 +615 @@
-	AddScore ((nCombos(nCurrentPlayer)*cScoreCombo) + (nPlanets(nCurrentPlayer)*cScorePlanet)) * nBonusMultiplier(nCurrentPlayer)
+	AddScore (((nCombos(nCurrentPlayer)*cScoreCombo) + (nPlanets(nCurrentPlayer)*cScorePlanet)) * nBonusMultiplier(nCurrentPlayer))
@@ -3684 +3684 @@
-		GIOverhead.state = off
+		GIOverhead.state = 0
