--- Laser War (Data East 1987) w VR Room v2.0.vbs.original
+++ Laser War (Data East 1987) w VR Room v2.0.vbs
@@ -1412 +1412 @@
-	If isGIOn <> Not IsOff Then
+	If isGIOn <> (Not IsOff) Then
@@ -2673 +2673 @@
-		dim str, b, highestID
+		dim str, b, AllBalls, highestID : allBalls = getballs
@@ -2675 +2675 @@
-		for each b in gBOT
+		for each b in allBalls
@@ -2683 +2683 @@
-		for each b in gBOT
+		for each b in allBalls
@@ -2993,0 +2994,25 @@
+Class StandupTarget
+  Private m_primary, m_prim, m_sw, m_animate
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public default Function init(primary, prim, sw, animate)
+    Set m_primary = primary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+
+    Set Init = Me
+  End Function
+End Class
+
@@ -3015,9 +3040,9 @@
-ST22 = Array(sw22, psw22,22, 0)
-ST23 = Array(sw23, psw23,23, 0)
-ST24 = Array(sw24, psw24,24, 0)
-ST30 = Array(sw30, psw30,30, 0)
-ST31 = Array(sw31, psw31,31, 0)
-ST32 = Array(sw32, psw32,32, 0)
-ST35 = Array(sw35, psw35,35, 0)
-ST36 = Array(sw36, psw36,36, 0)
-ST37 = Array(sw37, psw37,37, 0)
+Set ST22 = (new StandupTarget)(sw22, psw22,22, 0)
+Set ST23 = (new StandupTarget)(sw23, psw23,23, 0)
+Set ST24 = (new StandupTarget)(sw24, psw24,24, 0)
+Set ST30 = (new StandupTarget)(sw30, psw30,30, 0)
+Set ST31 = (new StandupTarget)(sw31, psw31,31, 0)
+Set ST32 = (new StandupTarget)(sw32, psw32,32, 0)
+Set ST35 = (new StandupTarget)(sw35, psw35,35, 0)
+Set ST36 = (new StandupTarget)(sw36, psw36,36, 0)
+Set ST37 = (new StandupTarget)(sw37, psw37,37, 0)
@@ -3048 +3073 @@
-	STArray(i)(3) =  STCheckHit(Activeball,STArray(i)(0))
+	STArray(i).animate =  STCheckHit(Activeball,STArray(i).primary)
@@ -3050,2 +3075,2 @@
-	If STArray(i)(3) <> 0 Then
-		DTBallPhysics Activeball, STArray(i)(0).orientation, STMass
+	If STArray(i).animate <> 0 Then
+		DTBallPhysics Activeball, STArray(i).primary.orientation, STMass
@@ -3059 +3084 @@
-		If STArray(i)(2) = switch Then STArrayID = i:Exit Function 
+		If STArray(i).sw = switch Then STArrayID = i:Exit Function 
@@ -3088 +3113 @@
-		STArray(i)(3) = STAnimate(STArray(i)(0),STArray(i)(1),STArray(i)(2),STArray(i)(3))
+		STArray(i).animate = STAnimate(STArray(i).primary,STArray(i).prim,STArray(i).sw,STArray(i).animate)
