--- Darkest Dungeon pupevent_2.3c.vbs.original
+++ Darkest Dungeon pupevent_2.3c.vbs
@@ -178,5 +178,5 @@
-Dim pupPackScreenFile: pupPackScreenFile = PuPlayer.GetRoot & "DarkestDungeon\ScreenType.txt"
-Dim ObjFso: Set ObjFso = CreateObject("Scripting.FileSystemObject")
-Dim ObjFile: Set ObjFile = ObjFso.OpenTextFile(pupPackScreenFile)
-Dim DMDType: DMDType = ObjFile.ReadLine
-'Dim DMDType: DMDType = 2
+' Dim pupPackScreenFile: pupPackScreenFile = PuPlayer.GetRoot & "DarkestDungeon\ScreenType.txt"
+' Dim ObjFso: Set ObjFso = CreateObject("Scripting.FileSystemObject")
+' Dim ObjFile: Set ObjFile = ObjFso.OpenTextFile(pupPackScreenFile)
+' Dim DMDType: DMDType = ObjFile.ReadLine
+Dim DMDType: DMDType = 2
