--- DarkPrincess1.3.1.vbs.original
+++ DarkPrincess1.3.1.vbs
@@ -164 +164 @@
-Playmusic "DPPlay.Mp3",Mlevel
+Playmusic "DPPlay.mp3",Mlevel
@@ -279 +279 @@
-Playmusic "DPELSA.Mp3",Mlevel
+Playmusic "DPELSA.mp3",Mlevel
@@ -324 +324 @@
-Playmusic "DPELSA.Mp3",Mlevel
+Playmusic "DPELSA.mp3",Mlevel
@@ -332 +332 @@
-Playmusic "DPPlay.Mp3",Mlevel
+Playmusic "DPPlay.mp3",Mlevel
@@ -1103 +1103 @@
-		if background ="" or len(background) <3 then background="Blank.png"
+		if background ="" or len(background) <3 then background="blank.png"
@@ -1148 +1148 @@
-        UltraDMD.DisplayScene00 "ballsave.gif", "", 0, "", 0, 14, GifTime1, 14
+        UltraDMD.DisplayScene00 "Ballsave.gif", "", 0, "", 0, 14, GifTime1, 14
@@ -1315,6 +1315,6 @@
-		DMD2 "Blank.png",  "","FUN  Select"  , 500 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "",""  , 50 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "","Fun  Select"  , 500 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "",""  , 50 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "","FUN  Select"  , 500 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "",""  , 50 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "","FUN  Select"  , 500 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "",""  , 50 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "","Fun  Select"  , 500 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "",""  , 50 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "","FUN  Select"  , 500 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "",""  , 50 , UltraDmd_eNone , ""
@@ -1346,6 +1346,6 @@
-		DMD2 "Blank.png",  "","Pro  Select"  , 500 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "",""  , 50 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "","Pro  Select"  , 500 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "",""  , 50 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "","Pro  Select"  , 500 , UltraDmd_eNone , ""
-		DMD2 "Blank.png",  "",""  , 50 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "","Pro  Select"  , 500 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "",""  , 50 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "","Pro  Select"  , 500 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "",""  , 50 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "","Pro  Select"  , 500 , UltraDmd_eNone , ""
+		DMD2 "blank.png",  "",""  , 50 , UltraDmd_eNone , ""
@@ -1381,4 +1381,4 @@
-	DMD2 "Blank.png",  "1 - " + HighScoreName(0),HighScore(0)  , 3000 , UltraDmd_eNone , ""
-	DMD2 "Blank.png",  "2 - " + HighScoreName(1),HighScore(1)  , 3000 , UltraDmd_eNone , ""
-	DMD2 "Blank.png",  "3 - " + HighScoreName(2),HighScore(2)  , 3000 , UltraDmd_eNone , ""
-	DMD2 "Blank.png",  "4 - " + HighScoreName(3),HighScore(3)  , 3000 , UltraDmd_eNone , ""	
+	DMD2 "blank.png",  "1 - " + HighScoreName(0),HighScore(0)  , 3000 , UltraDmd_eNone , ""
+	DMD2 "blank.png",  "2 - " + HighScoreName(1),HighScore(1)  , 3000 , UltraDmd_eNone , ""
+	DMD2 "blank.png",  "3 - " + HighScoreName(2),HighScore(2)  , 3000 , UltraDmd_eNone , ""
+	DMD2 "blank.png",  "4 - " + HighScoreName(3),HighScore(3)  , 3000 , UltraDmd_eNone , ""	
@@ -1403,2 +1403,2 @@
-    DMD2 "Blank.png",  "Select FUN For","Ramdom Missions"  , 3000 , UltraDmd_eNone , ""
-	DMD2 "Blank.png",  "Select PRO For","Progress Missions"  , 3000 , UltraDmd_eNone , ""
+    DMD2 "blank.png",  "Select FUN For","Ramdom Missions"  , 3000 , UltraDmd_eNone , ""
+	DMD2 "blank.png",  "Select PRO For","Progress Missions"  , 3000 , UltraDmd_eNone , ""
@@ -1772,15 +1772,15 @@
-GetDMDColor
-Sub GetDMDColor
-Dim WshShell,filecheck,directory
-Set WshShell = CreateObject("WScript.Shell")
-If DMDSize then
-WshShell.RegWrite "HKCU\Software\UltraDMD\w",DMDWidth,"REG_DWORD"
-WshShell.RegWrite "HKCU\Software\UltraDMD\h",DMDHeight,"REG_DWORD"
-End if
-If DMDPosition then
-WshShell.RegWrite "HKCU\Software\UltraDMD\x",DMDPosX,"REG_DWORD"
-WshShell.RegWrite "HKCU\Software\UltraDMD\y",DMDPosY,"REG_DWORD"
-End if
-WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
-WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
-End Sub
+' GetDMDColor
+' Sub GetDMDColor
+' Dim WshShell,filecheck,directory
+' Set WshShell = CreateObject("WScript.Shell")
+' If DMDSize then
+' WshShell.RegWrite "HKCU\Software\UltraDMD\w",DMDWidth,"REG_DWORD"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\h",DMDHeight,"REG_DWORD"
+' End if
+' If DMDPosition then
+' WshShell.RegWrite "HKCU\Software\UltraDMD\x",DMDPosX,"REG_DWORD"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\y",DMDPosY,"REG_DWORD"
+' End if
+' WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor",UseFullColor,"REG_SZ"
+' WshShell.RegWrite "HKCU\Software\UltraDMD\color",DMDColorSelect,"REG_SZ"
+' End Sub
@@ -1838 +1838 @@
-			DMD2 "Blank.png",  "4 - " + HighScoreName(3),HighScore(3)  , 3000 , UltraDmd_eNone , ""	
+			DMD2 "blank.png",  "4 - " + HighScoreName(3),HighScore(3)  , 3000 , UltraDmd_eNone , ""	
