--- ./Indiana Jones The Pinball Adventure (Williams 1993) VPWmod v1.1/Indiana Jones The Pinball Adventure (Williams 1993) VPWmod v1.1.vbs.original	2023-06-02 16:12:14
+++ ./Indiana Jones The Pinball Adventure (Williams 1993) VPWmod v1.1/Indiana Jones The Pinball Adventure (Williams 1993) VPWmod v1.1.vbs	2023-06-02 16:12:01
@@ -4357,9 +4357,9 @@
 	i = DTArrayID(switch)
 
 '	PlaySoundAtVol  DTHitSound, Activeball, Vol(Activeball)*22.5
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
 	End If
 	DoDTAnim
 End Sub
\ No newline at end of file
@@ -4368,7 +4368,7 @@
 	Dim i
 	i = DTArrayID(switch)
 
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
 	DoDTAnim
 End Sub
 
\ No newline at end of file
@@ -4376,14 +4376,14 @@
 	Dim i
 	i = DTArrayID(switch)
 
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
 	DoDTAnim
 End Sub
 
 Function DTArrayID(switch)
 	Dim i
 	For i = 0 to uBound(DTArray) 
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
 	Next
 End Function
 
\ No newline at end of file
@@ -4492,7 +4492,7 @@
 Sub DoDTAnim()
 	Dim i
 	For i=0 to Ubound(DTArray)
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
 	Next
 End Sub
 
\ No newline at end of file
@@ -4698,7 +4698,40 @@
 '******************************************************
 '		DROP TARGETS INITIALIZATION
 '******************************************************
+
+Class DropTarget
+  Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+  Public Property Let Secondary(input): Set m_secondary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
 
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+  Public Property Let IsDropped(input): m_isDropped = input: End Property
+
+  Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+    Set m_primary = primary
+    Set m_secondary = secondary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+    m_isDropped = isDropped
+
+    Set Init = Me
+  End Function
+End Class
+
 'Define a variable for each drop target
 Dim DT115, DT116, DT117, DT11
 
\ No newline at end of file
@@ -4717,11 +4750,12 @@
 '	animate:			Arrary slot for handling the animation instrucitons, set to 0
 '
 '	Values for annimate: 1 - bend target (hit to primary), 2 - drop target (hit to secondary), 3 - brick target (high velocity hit to secondary), -1 - raise target 
+'       isDropped:  Boolean which determines whether a drop target is dropped. Set to false if they are initially raised, true if initially dropped.
 
-DT11 = Array(sw11a,sw11b, p_sw11, 11, 0)
-DT115 = Array(sw115a,sw115b, p_sw115, 115, 0)
-DT116 = Array(sw116a,sw116b, p_sw116, 116, 0)
-DT117 = Array(sw117a,sw117b, p_sw117, 117, 0)
+Set DT11 = (new DropTarget)(sw11a,sw11b, p_sw11, 11, 0, false)
+Set DT115 = (new DropTarget)(sw115a,sw115b, p_sw115, 115, 0, false)
+Set DT116 = (new DropTarget)(sw116a,sw116b, p_sw116, 116, 0, false)
+Set DT117 = (new DropTarget)(sw117a,sw117b, p_sw117, 117, 0, false)
 
 'Add all the Drop Target Arrays to Drop Target Animation Array
 ' DTAnimationArray = Array(DT1, DT2, ....)
\ No newline at end of file
