--- SPP.vbs.original	2024-03-09 09:57:55.086385066 +0100
+++ SPP.vbs	2024-03-09 09:53:27.910563652 +0100
@@ -2185,7 +2185,7 @@
                 DMDScene.GetImage("Dig" & i).SetBounds 4 + i * 6, 3 + 16 + 2, 8, 8
             Next
             For i = 20 to 35 ' Bottom
-                DMDScene.GetImage("Dig" & i).SetBounds (i - 20) * 8, 3, 8, 16
+                DMDScene.GetImage("Dig" & i).SetBounds 8 * (i - 20), 3, 8, 16
             Next
             FlexDMD.LockRenderThread
             FlexDMD.Stage.AddActor DMDScene
