--- Surf'n Safari v1.3.4.vbs.original
+++ Surf'n Safari v1.3.4.vbs
@@ -119,0 +120,4 @@
+'************************ DT VpinMame or LedDMD visible: 0 VpinMame visible , 1 LedDMD visible
+
+Const LedDMDDt = 1
+
@@ -126,4 +129,0 @@
-'************************ DT VpinMame or LedDMD visible: 0 VpinMame visible , 1 LedDMD visible
-
-Const LedDMDDt = 1
-
@@ -276 +276 @@
-	Trim.visible=1
+	'Trim.visible=1
@@ -309 +309 @@
-	Trim.visible=RailsLights
+	'Trim.visible=RailsLights
