--- Alfred Hitchcock's Psycho (TBA 2019).vbs.original
+++ Alfred Hitchcock's Psycho (TBA 2019).vbs
@@ -295,5 +295,5 @@
-SolCallback(7) = "dtbank1..SolHit 1,"
-SolCallback(8) = "dtbank1..SolHit 2,"
-SolCallback(9) = "dtbank1..SolHit 3,"
-SolCallback(10) = "dtbank1..SolHit 4,"
-SolCallback(11) = "dtbank1..SolHit 5,"
+SolCallback(7) = "dtbank1.SolHit 1,"
+SolCallback(8) = "dtbank1.SolHit 2,"
+SolCallback(9) = "dtbank1.SolHit 3,"
+SolCallback(10) = "dtbank1.SolHit 4,"
+SolCallback(11) = "dtbank1.SolHit 5,"
