--- ./BarbWire(Gottlieb1996)JoePicassoModv1.2/BarbWire(Gottlieb1996)JoePicassoModv1.2.vbs.original	2023-11-22 07:09:00
+++ ./BarbWire(Gottlieb1996)JoePicassoModv1.2/BarbWire(Gottlieb1996)JoePicassoModv1.2.vbs	2023-11-22 07:08:56
@@ -265,7 +265,7 @@
 		.HandleMechanics=0
 		.ShowFrame=0
 		.Games(cGameName).Settings.Value("rol") = 0
-		if Mute_Sound_For_PuPPack = 1 then.Games(cGameName).Settings.Value("sound") = 0
+		if Mute_Sound_For_PuPPack = 1 then .Games(cGameName).Settings.Value("sound") = 0
 		.Run GetPlayerHwnd
 		If Err Then MsgBox Err.Description
 	End With
@@ -2668,4 +2668,4 @@
 End Sub
 ' *********************************************************************
 '                 End Swap Plastics and Flippers Functions
-' *********************************************************************
\ No newline at end of file
+' *********************************************************************
