--- "Munsters (Original 2020) 1.05.vbs.original"	2024-05-30 15:48:07.698227430 +0200
+++ "Munsters (Original 2020) 1.05.vbs"	2024-05-30 15:46:42.828529130 +0200
@@ -257,8 +257,9 @@
 '============================
 
 Dim osbid:      osbid="MMadness"                             ' your orbital scoreboard login name or use the default MMadness code
-Dim osbkey:    osbkey="09dcddca-8710-11ea-97f8-42010a8a06b6" ' your orbital scoreboard api key or use the default Munster Table code
-
+'Dim osbkey:    osbkey="09dcddca-8710-11ea-97f8-42010a8a06b6" ' your orbital scoreboard api key or use the default Munster Table code
+' Orbital score board is dead
+Dim osbkey: osbkey=""
 
 
 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
@@ -6014,7 +6015,7 @@
 				PuPlayer.playstop pAudio
 				PuPlayer.playlistplayex pAudio,"audionoise","Sound-0x018A.mp3",90,90
 				MovieText CenterText(24,"TOTAL BONUS"), CenterText(26,FormatNumber( (50000+(10000*AreasComplete))*BonusMultiplier(CurrentPlayer),0))
-				AddScore (50000+(10000*AreasComplete))*BonusMultiplier(CurrentPlayer)
+				AddScore ((50000+(10000*AreasComplete))*BonusMultiplier(CurrentPlayer))
 			Case 26
 				ClearPage3Timer.interval = 10
 				ClearPage3Timer.Enabled = True
