--- Munsters (Original 2020) 1.05.vbs.original
+++ Munsters (Original 2020) 1.05.vbs
@@ -260,2 +260,3 @@
-Dim osbkey:    osbkey="09dcddca-8710-11ea-97f8-42010a8a06b6" ' your orbital scoreboard api key or use the default Munster Table code
-
+'Dim osbkey:    osbkey="09dcddca-8710-11ea-97f8-42010a8a06b6" ' your orbital scoreboard api key or use the default Munster Table code
+' Orbital score board is dead
+Dim osbkey: osbkey=""
@@ -6017 +6018 @@
-				AddScore (50000+(10000*AreasComplete))*BonusMultiplier(CurrentPlayer)
+				AddScore ((50000+(10000*AreasComplete))*BonusMultiplier(CurrentPlayer))
