--- 2104398928_CARtoonsRC(Nailed2021)v1.3.vbs.original
+++ 2104398928_CARtoonsRC(Nailed2021)v1.3.vbs
@@ -844 +844,3 @@
-			dy=-1*(EVAL("roachxy"&xx)(1)(roachstep)-EVAL("roachxy"&xx)(1)(roachstep-1))	'delta Y
+			dim roachxy : roachxy = EVAL("roachxy"&xx)
+			'dy=-1*(EVAL("roachxy"&xx)(1)(roachstep)-EVAL("roachxy"&xx)(1)(roachstep-1))	'delta Y
+			dy=-1*(roachxy(1)(roachstep)-roachxy(1)(roachstep-1))	'delta Y
@@ -846 +848,2 @@
-			dx=EVAL("roachxy"&xx)(0)(roachstep)-EVAL("roachxy"&xx)(0)(roachstep-1)		'delta X
+			'dx=EVAL("roachxy"&xx)(0)(roachstep)-EVAL("roachxy"&xx)(0)(roachstep-1)		'delta X
+			dx=roachxy(0)(roachstep)-roachxy(0)(roachstep-1)		'delta X
@@ -1889 +1892 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1907 +1910 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1921 +1924 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1935 +1938 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1949 +1952 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1963 +1966 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1978 +1981 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
@@ -1993 +1996 @@
-	addscore (starstate+1)*1000
+	addscore 1000*(starstate+1)
