--- ABBAv2.0.vbs.original
+++ ABBAv2.0.vbs
@@ -50 +50 @@
-	NVramPatchLoad
+	'NVramPatchLoad
@@ -116 +116,4 @@
-Sub table1_exit:NVramPatchExit:Controller.stop:End Sub
+Sub table1_exit
+    'NVramPatchExit
+    Controller.stop
+End Sub
@@ -305,5 +308,5 @@
-SolCallback(7) = "dtbank1..SolHit 1,"
-SolCallback(8) = "dtbank1..SolHit 2,"
-SolCallback(9) = "dtbank1..SolHit 3,"
-SolCallback(10) = "dtbank1..SolHit 4,"
-SolCallback(11) = "dtbank1..SolHit 5,"
+SolCallback(7) = "dtbank1.SolHit 1,"
+SolCallback(8) = "dtbank1.SolHit 2,"
+SolCallback(9) = "dtbank1.SolHit 3,"
+SolCallback(10) = "dtbank1.SolHit 4,"
+SolCallback(11) = "dtbank1.SolHit 5,"
@@ -470 +473 @@
-    NVramPatchKeyCheck
+    'NVramPatchKeyCheck
@@ -907,85 +910,85 @@
-' =============================================================================================================
-'                 NVram patch for Taito do Brasil tables by Pmax65
-'
-' NVramPatchExit	' Must be placed before the Controler.Stop statement into the Table1_Exit Sub
-' NVramPatchLoad	' Must be placed before the VPinMAME controller initialization
-' NVramPatchKeyCheck' Must be placed in the lamptimer timer
-' =============================================================================================================
- 
-Const GameOverLampID = 149 ' set this constant to the ID number of the game-over lamp
-Dim NVramPatchCoinCnt
-
-Function GetNVramPath()
-    Dim WshShell
-    Set WshShell = CreateObject("WScript.Shell")
-    GetNVramPath = WshShell.RegRead("HKEY_CURRENT_USER\Software\Freeware\Visual PinMame\globals\nvram_directory")
-End function
-
-Function FileExists(FileName)
-    DIM FSO
-    FileExists = False
-    Set FSO = CreateObject("Scripting.FileSystemObject")
-    FileExists = FSO.FileExists(FileName)
-    Set FSO = Nothing
-End Function
-
-Sub Kill(FileName)
-    Dim ObjFile, FSO
-    On Error Resume Next
-    Set FSO = CreateObject("Scripting.FileSystemObject")
-    Set ObjFile = FSO.GetFile(FileName)
-    ObjFile.Delete
-    On Error Goto 0
-    Set FSO = Nothing
-End Sub
-
-Sub Copy(SourceFileName, DestFileName)
-    Dim FSO
-    On Error Resume Next
-    Set FSO = CreateObject("Scripting.FileSystemObject")
-    FSO.CopyFile SourceFileName, DestFileName, True
-    On Error Goto 0
-    Set FSO = Nothing
-End Sub
-
-Sub NVramPatchLoad
-    NVramPatchCoinCnt = 0
-    If FileExists(GetNVramPath + "\" + cGameName + ".nvb")Then
-        Copy GetNVramPath + "\" + cGameName + ".nvb", GetNVramPath + "\" + cGameName + ".nv"
-    Else
-        Copy GetNVramPath + "\" + cGameName + ".nv", GetNVramPath + "\" + cGameName + ".nvb"
-    End If
-End Sub
-
-Sub NVramPatchExit
-     
-        Kill GetNVramPath + "\" + cGameName + ".nvb"
-        Do
-    
-		LampTimer_Timer          ' This loop is needed to avoid the NVram reset (losing the hi-score and credits)
-        Loop Until LampState(20) = 1 ' when the game is over but the match procedure isn't still ended
-
-End Sub
-' =============================================================================================================
-' To completely erase the NVram file keep the Start Game button pushed while inserting
-' two coins into the first coin slit (this resets the high scores too)
-' =============================================================================================================
-
-Sub NVramPatchKeyCheck
-    If Controller.Switch(swStartButton)then
-        If Controller.Switch(swCoin1)then
-            If NVramPatchCoinCnt = 2 Then
-                Controller.Stop
-                Kill GetNVramPath + "\" + cGameName + ".nv"
-                Kill GetNVramPath + "\" + cGameName + ".nvb"
-                QuitPlayer 2
-            Else
-                NVramPatchCoinCnt = 1
-            End If
-        Else
-            If NVramPatchCoinCnt = 1 Then
-                NVramPatchCoinCnt = 2
-            End If
-        End If
-    Else
-        NVramPatchCoinCnt = 0
+' ' =============================================================================================================
+' '                 NVram patch for Taito do Brasil tables by Pmax65
+' '
+' ' NVramPatchExit	' Must be placed before the Controler.Stop statement into the Table1_Exit Sub
+' ' NVramPatchLoad	' Must be placed before the VPinMAME controller initialization
+' ' NVramPatchKeyCheck' Must be placed in the lamptimer timer
+' ' =============================================================================================================
+ 
+' Const GameOverLampID = 149 ' set this constant to the ID number of the game-over lamp
+' Dim NVramPatchCoinCnt
+
+' Function GetNVramPath()
+'     Dim WshShell
+'     Set WshShell = CreateObject("WScript.Shell")
+'     GetNVramPath = WshShell.RegRead("HKEY_CURRENT_USER\Software\Freeware\Visual PinMame\globals\nvram_directory")
+' End function
+
+' Function FileExists(FileName)
+'     DIM FSO
+'     FileExists = False
+'     Set FSO = CreateObject("Scripting.FileSystemObject")
+'     FileExists = FSO.FileExists(FileName)
+'     Set FSO = Nothing
+' End Function
+
+' Sub Kill(FileName)
+'     Dim ObjFile, FSO
+'     On Error Resume Next
+'     Set FSO = CreateObject("Scripting.FileSystemObject")
+'     Set ObjFile = FSO.GetFile(FileName)
+'     ObjFile.Delete
+'     On Error Goto 0
+'     Set FSO = Nothing
+' End Sub
+
+' Sub Copy(SourceFileName, DestFileName)
+'     Dim FSO
+'     On Error Resume Next
+'     Set FSO = CreateObject("Scripting.FileSystemObject")
+'     FSO.CopyFile SourceFileName, DestFileName, True
+'     On Error Goto 0
+'     Set FSO = Nothing
+' End Sub
+
+' Sub NVramPatchLoad
+'     NVramPatchCoinCnt = 0
+'     If FileExists(GetNVramPath + "\" + cGameName + ".nvb")Then
+'         Copy GetNVramPath + "\" + cGameName + ".nvb", GetNVramPath + "\" + cGameName + ".nv"
+'     Else
+'         Copy GetNVramPath + "\" + cGameName + ".nv", GetNVramPath + "\" + cGameName + ".nvb"
+'     End If
+' End Sub
+
+' Sub NVramPatchExit
+     
+'         Kill GetNVramPath + "\" + cGameName + ".nvb"
+'         Do
+    
+' 		LampTimer_Timer          ' This loop is needed to avoid the NVram reset (losing the hi-score and credits)
+'         Loop Until LampState(20) = 1 ' when the game is over but the match procedure isn't still ended
+
+' End Sub
+' ' =============================================================================================================
+' ' To completely erase the NVram file keep the Start Game button pushed while inserting
+' ' two coins into the first coin slit (this resets the high scores too)
+' ' =============================================================================================================
+
+' Sub NVramPatchKeyCheck
+'     If Controller.Switch(swStartButton)then
+'         If Controller.Switch(swCoin1)then
+'             If NVramPatchCoinCnt = 2 Then
+'                 Controller.Stop
+'                 Kill GetNVramPath + "\" + cGameName + ".nv"
+'                 Kill GetNVramPath + "\" + cGameName + ".nvb"
+'                 QuitPlayer 2
+'             Else
+'                 NVramPatchCoinCnt = 1
+'             End If
+'         Else
+'             If NVramPatchCoinCnt = 1 Then
+'                 NVramPatchCoinCnt = 2
+'             End If
+'         End If
+'     Else
+'         NVramPatchCoinCnt = 0
@@ -993,2 +996,2 @@
-	End If
-End Sub
+' 	End If
+' End Sub
