--- Cirqus_Voltaire_Hanibal-Mod_3.7.vbs.original
+++ Cirqus_Voltaire_Hanibal-Mod_3.7.vbs
@@ -1285 +1285 @@
-On Error Resume Next
+
@@ -1288 +1288 @@
-	Execute "Set Lights(" & i & ")  = L" & i
+	Execute "On Error Resume Next:Set Lights(" & i & ")  = L" & i & ":On Error Goto 0"
@@ -2667 +2667 @@
-Sub cSSHoles_Hit(idx):SubwayHandler Me(idx), 46:End Sub
+Sub cSSHoles_Hit(idx):SubwayHandler cSSHoles(idx), 46:End Sub
@@ -2670 +2670 @@
-Sub cRMHoles_Hit(idx):SubwayHandler Me(idx), 47:End Sub
+Sub cRMHoles_Hit(idx):SubwayHandler cRMHoles(idx), 47:End Sub
@@ -2690 +2690 @@
-Set WBall = kicker1.Createsizedball(51):WBall.color = (wbcolor):Wball.image = "powerball4":Wball.id = 666:kicker1.Kick 0, 0   
+Set WBall = kicker1.Createsizedball(51):WBall.color = (wbcolor):Wball.image = "powerball4":Wball.UserValue = 666:kicker1.Kick 0, 0   
@@ -3123 +3123 @@
-Set WBall = kicker1.Createsizedball(51):WBall.color = (wbcolor):Wball.image = "powerball4":Wball.id = 666:kicker1.Kick 0, 0
+Set WBall = kicker1.Createsizedball(51):WBall.color = (wbcolor):Wball.image = "powerball4":Wball.UserValue = 666:kicker1.Kick 0, 0
@@ -3166 +3166 @@
-f24scale = 0
+f24gscale = 0
@@ -3189 +3189 @@
-f21scale = 0
+f21gscale = 0
@@ -3212 +3212 @@
-f25scale = 0
+f25gscale = 0
@@ -3235 +3235 @@
-f26scale = 0
+f26gscale = 0
@@ -3258 +3258 @@
-f37scale = 0
+f37gscale = 0
