--- Twilight Zone (Bally 1993) 2.3.6.vbs.original
+++ Twilight Zone (Bally 1993) 2.3.6.vbs
@@ -2884,6 +2884,31 @@
 '		STAND-UP TARGET INITIALIZATION
 '******************************************************
 
+Class StandupTarget
+  Private m_primary, m_prim, m_sw, m_animate
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public default Function init(primary, prim, sw, animate)
+    Set m_primary = primary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+
+    Set Init = Me
+  End Function
+End Class
+
 'Define a variable for each stand-up target
 Dim ST47, ST48, ST64, ST65, ST65a, ST66, ST67, ST68, ST77, ST78
 
@@ -2897,16 +2922,16 @@
 '	switch:				ROM switch number
 '	animate:			Arrary slot for handling the animation instrucitons, set to 0
 
-ST47 = Array(sw47, sw47_BM_Dark_Room, 47, 0)
-ST48 = Array(sw48, sw48_BM_Dark_Room ,48, 0)
-ST64 = Array(sw64, sw64_BM_Dark_Room,64, 0)
-ST65 = Array(sw65, sw65_BM_Dark_Room,65, 0)
-ST65a = Array(sw65a, sw65a_BM_Dark_Room,165, 0)
-ST66 = Array(sw66, sw66_BM_Dark_Room,66, 0)
-ST67 = Array(sw67, sw67_BM_Dark_Room,67, 0)
-ST68 = Array(sw68, sw68_BM_Dark_Room,68, 0)
-ST77 = Array(sw77, sw77_BM_Dark_Room,77, 0)
-ST78 = Array(sw78, sw78_BM_Dark_Room,78, 0)
+Set ST47 = (new StandupTarget)(sw47, sw47_BM_Dark_Room, 47, 0)
+Set ST48 = (new StandupTarget)(sw48, sw48_BM_Dark_Room ,48, 0)
+Set ST64 = (new StandupTarget)(sw64, sw64_BM_Dark_Room,64, 0)
+Set ST65 = (new StandupTarget)(sw65, sw65_BM_Dark_Room,65, 0)
+Set ST65a = (new StandupTarget)(sw65a, sw65a_BM_Dark_Room,165, 0)
+Set ST66 = (new StandupTarget)(sw66, sw66_BM_Dark_Room,66, 0)
+Set ST67 = (new StandupTarget)(sw67, sw67_BM_Dark_Room,67, 0)
+Set ST68 = (new StandupTarget)(sw68, sw68_BM_Dark_Room,68, 0)
+Set ST77 = (new StandupTarget)(sw77, sw77_BM_Dark_Room,77, 0)
+Set ST78 = (new StandupTarget)(sw78, sw78_BM_Dark_Room,78, 0)
 
 
 
@@ -2931,10 +2956,10 @@
 	i = STArrayID(switch)
 
 	PlayTargetSound
-	STArray(i)(3) =  STCheckHit(Activeball,STArray(i)(0))
+	STArray(i).animate =  STCheckHit(Activeball,STArray(i).primary)
 
-	If STArray(i)(3) <> 0 Then
-		STBallPhysics Activeball, STArray(i)(0).orientation, STMass
+	If STArray(i).animate <> 0 Then
+		STBallPhysics Activeball, STArray(i).primary.orientation, STMass
 	End If
 	DoSTAnim
 End Sub
@@ -2942,7 +2967,7 @@
 Function STArrayID(switch)
 	Dim i
 	For i = 0 to uBound(STArray) 
-		If STArray(i)(2) = switch Then STArrayID = i:Exit Function 
+		If STArray(i).sw = switch Then STArrayID = i:Exit Function 
 	Next
 End Function
 
@@ -2984,7 +3009,7 @@
 Sub DoSTAnim()
 	Dim i
 	For i=0 to Ubound(STArray)
-		STArray(i)(3) = STAnimate(STArray(i)(0),STArray(i)(1),STArray(i)(2),STArray(i)(3))
+		STArray(i).animate = STAnimate(STArray(i).primary,STArray(i).prim,STArray(i).sw,STArray(i).animate)
 	Next
 
 	sw47_LM_Flashers_f18.transy = sw47_BM_Dark_Room.transy
