--- "Darkest Dungeon pupevent_2.2.vbs.original"	2024-06-05 13:57:07.904591501 +0200
+++ "Darkest Dungeon pupevent_2.2.vbs"	2024-06-05 13:56:39.381673139 +0200
@@ -98,11 +98,11 @@
 '	Table1_exit
 'End If
 
-Dim pupPackScreenFile: pupPackScreenFile = PuPlayer.GetRoot & "DarkestDungeon\ScreenType.txt"
-Dim ObjFso: Set ObjFso = CreateObject("Scripting.FileSystemObject")
-Dim ObjFile: Set ObjFile = ObjFso.OpenTextFile(pupPackScreenFile)
-Dim DMDType: DMDType = ObjFile.ReadLine
-'Dim DMDType: DMDType = 2
+'Dim pupPackScreenFile: pupPackScreenFile = PuPlayer.GetRoot & "DarkestDungeon\ScreenType.txt"
+'Dim ObjFso: Set ObjFso = CreateObject("Scripting.FileSystemObject")
+'Dim ObjFile: Set ObjFile = ObjFso.OpenTextFile(pupPackScreenFile)
+'Dim DMDType: DMDType = ObjFile.ReadLine
+Dim DMDType: DMDType = 2
 
 if DMDType = 2 Then
 	PupScreen1 = 0 ' Do not use apron pupscreens
