--- ./Game of Thrones LE (Stern 2015) VPW v1.0.2/Game of Thrones LE (Stern 2015) VPW v1.0.2.vbs	2023-11-28 07:36:21
+++ ./Game of Thrones LE (Stern 2015) VPW v1.0.2/Game of Thrones LE (Stern 2015) VPW v1.0.2.vbs.dmd	2023-11-28 07:36:21
@@ -163,8 +163,19 @@
 Dim Topper: Topper = 1			     '0 - VR Topper Off  1 - VR Topper On
 '**************************
 
+
 '/////// No User Configurable options beyond this point ////////
 
+'Hack to display the DMD on Standalone 
+Sub ImplicitDMD_Init
+   Me.x = 30
+   Me.y = 30
+   Me.width = 128 * 2
+   Me.height = 32 * 2
+   Me.visible = true
+End Sub
+' End of Hack to display the DMD on standalone version
+
 If RenderingMode = 2 Then VRRoom = VRRoomChoice Else VRRoom = 0
 Dim DesktopMode: DesktopMode = table1.ShowDT
 
@@ -5018,6 +5029,14 @@
     LeftUFlipperTop.RotZ = LeftUFlipper.CurrentAngle
     RightUFlipperTop.RotZ = RightUFlipper.CurrentAngle
 ' add any other real time update subs, like gates or diverters, flippers
+' Hack to display the DMD on standalone version
+    Dim DMDp: DMDp = FlexDMD.DmdPixels
+    If Not IsEmpty(DMDp) Then
+          DMDWidth = FlexDMD.Width
+          DMDHeight = FlexDMD.Height
+          DMDPixels = DMDp
+    End If
+' End of Hack to display the DMD on standalone version
 End Sub
 
 
