--- Truck Stop (Bally 1988) v2.1.0.vbs.original
+++ Truck Stop (Bally 1988) v2.1.0.vbs
@@ -287,10 +286,0 @@
-'****								Knocker               						****
-'***********************************************************************************
-SolCallback(sKnocker) = "HitKnocker"
-
-Sub HitKnocker(enabled)
-	If enabled Then PlaySound SoundFX("Knocker",DOFKnocker), 1
-End Sub
-
-
-'***********************************************************************************
@@ -307,0 +298,11 @@
+
+
+'***********************************************************************************
+'****								Knocker               						****
+'***********************************************************************************
+SolCallback(sKnocker) = "HitKnocker"
+
+Sub HitKnocker(enabled)
+	If enabled Then PlaySound SoundFX("Knocker",DOFKnocker), 1
+End Sub
+
