--- Ice Age Christmas (Original Balutito 2021) endeemillr mod.vbs.original
+++ Ice Age Christmas (Original Balutito 2021) endeemillr mod.vbs
@@ -154,4 +154 @@
-DIM music
-
-music = "iceage.mp3"
-PlayMusic "iceage.mp3" 
+PlayMusic "Music/iceage.mp3" 
@@ -160 +157 @@
-PlayMusic "iceage.mp3"
+PlayMusic "Music/iceage.mp3"
@@ -958,2 +955,3 @@
-	Set fso = CreateObject("Scripting.FileSystemObject")
-	curDir = fso.GetAbsolutePathName(".")
+	' Set fso = CreateObject("Scripting.FileSystemObject")
+	' curDir = fso.GetAbsolutePathName(".")
+    curDir = "."
