--- Simpsons Pinball Party, The (Stern 2003) VPW 2.0.3.vbs.original
+++ Simpsons Pinball Party, The (Stern 2003) VPW 2.0.3.vbs
@@ -3335,11 +3335,11 @@
-	Public Sub MassAssign(aIdx, aInput)
-		If typename(obj(aIdx)) = "NullFadingObject" Then 'if empty, use Set
-			if IsArray(aInput) then
-				obj(aIdx) = aInput
-			Else
-				Set obj(aIdx) = aInput
-			end if
-		Else
-			Obj(aIdx) = AppendArray(obj(aIdx), aInput)
-		end if	
-	end Sub
+'	Public Sub MassAssign(aIdx, aInput)
+'		If typename(obj(aIdx)) = "NullFadingObject" Then 'if empty, use Set
+'			if IsArray(aInput) then
+'				obj(aIdx) = aInput
+'			Else
+'				Set obj(aIdx) = aInput
+'			end if
+'		Else
+'			Obj(aIdx) = AppendArray(obj(aIdx), aInput)
+'		end if	
+'	end Sub
