--- ./Simpsons Pinball Party, The (Stern 2003) VPW 2.0.3/Simpsons Pinball Party, The (Stern 2003) VPW 2.0.3.vbs.original	2023-06-02 18:22:44
+++ ./Simpsons Pinball Party, The (Stern 2003) VPW 2.0.3/Simpsons Pinball Party, The (Stern 2003) VPW 2.0.3.vbs	2023-06-02 18:23:40
@@ -3332,17 +3332,17 @@
 	Public Property Get state(idx) : state = Lvl(idx) : end Property
 
 	'Mass assign, Builds arrays where necessary
-	Public Sub MassAssign(aIdx, aInput)
-		If typename(obj(aIdx)) = "NullFadingObject" Then 'if empty, use Set
-			if IsArray(aInput) then
-				obj(aIdx) = aInput
-			Else
-				Set obj(aIdx) = aInput
-			end if
-		Else
-			Obj(aIdx) = AppendArray(obj(aIdx), aInput)
-		end if	
-	end Sub
+'	Public Sub MassAssign(aIdx, aInput)
+'		If typename(obj(aIdx)) = "NullFadingObject" Then 'if empty, use Set
+'			if IsArray(aInput) then
+'				obj(aIdx) = aInput
+'			Else
+'				Set obj(aIdx) = aInput
+'			end if
+'		Else
+'			Obj(aIdx) = AppendArray(obj(aIdx), aInput)
+'		end if	
+'	end Sub
 	
 	Public Property Let MassAssign(aIdx, aInput)
 		If typename(obj(aIdx)) = "NullFadingObject" Then 'if empty, use Set
