--- "TRON Classic (Original 2022) v2.1.vbs.original"	2024-02-27 14:37:53.866319313 +0100
+++ "TRON Classic (Original 2022) v2.1.vbs"	2024-02-27 14:36:34.891954237 +0100
@@ -607,7 +607,7 @@
 
 'Table Init
 Sub table1_Init
-    vpmInit me:LoadLUT:NVOffset (13)
+    vpmInit me:LoadLUT':NVOffset (13)
     With Controller
         .GameName = cGameName
         If Err Then MsgBox "Can't start Game" & cGameName & vbNewLine & Err.Description:Exit Sub
