--- Monkey Island VR Room.vbs.original
+++ Monkey Island VR Room.vbs
@@ -243 +243 @@
-	UseFlexDMD = False
+	UseFlexDMD = True
@@ -2463 +2463 @@
-                DMDScene.GetImage("Dig" & i).SetBounds (i - dCharsPerLine(1)) * 8, 3, 8, 16
+                DMDScene.GetImage("Dig" & i).SetBounds 8 * (i - dCharsPerLine(1)), 3, 8, 16
@@ -4414 +4414 @@
-	Kicker002.CreateSizedBall(BallSize/2).ID = MyTroughLR(0)
+	Kicker002.CreateSizedBall(BallSize/2).UserValue = MyTroughLR(0)
@@ -4435 +4435 @@
-	Kicker006.CreateSizedBall(BallSize/2).ID = MyTroughTreasure(0)
+	Kicker006.CreateSizedBall(BallSize/2).UserValue = MyTroughTreasure(0)
@@ -4452 +4452 @@
-	Kicker003.CreateSizedBall(BallSize/2).ID = MyTroughLeChuck(0)
+	Kicker003.CreateSizedBall(BallSize/2).UserValue = MyTroughLeChuck(0)
@@ -4466 +4466 @@
-	Kicker007.CreateSizedBall(BallSize/2).ID = MyTroughTreasure(0)
+	Kicker007.CreateSizedBall(BallSize/2).UserValue = MyTroughTreasure(0)
@@ -4477 +4477 @@
-	MyTroughLR(UBound(MyTroughLR)) = tBall.ID
+	MyTroughLR(UBound(MyTroughLR)) = tBall.UserValue
@@ -4484 +4484 @@
-	MyTroughTreasure(Ubound(MyTroughTreasure)) = tBall.ID
+	MyTroughTreasure(Ubound(MyTroughTreasure)) = tBall.UserValue
@@ -4490 +4490 @@
-	MyTroughLeChuck(Ubound(MyTroughLeChuck)) = tBall.ID
+	MyTroughLeChuck(Ubound(MyTroughLeChuck)) = tBall.UserValue
