--- Thundercats Pinball v1.0.9.vbs.original
+++ Thundercats Pinball v1.0.9.vbs
@@ -66 +66 @@
-Dim PuPlayer: Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay") 
+'Dim PuPlayer: Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay") 
@@ -68,4 +68,4 @@
-Dim pupPackScreenFile: pupPackScreenFile = PuPlayer.GetRoot & "Thundercats\ScreenType.txt"
-Dim ObjFso: Set ObjFso = CreateObject("Scripting.FileSystemObject")
-Dim ObjFile: Set ObjFile = ObjFso.OpenTextFile(pupPackScreenFile)
-Dim DMDType: DMDType = ObjFile.ReadLine
+'Dim pupPackScreenFile: pupPackScreenFile = PuPlayer.GetRoot & "Thundercats\ScreenType.txt"
+'Dim ObjFso: Set ObjFso = CreateObject("Scripting.FileSystemObject")
+'Dim ObjFile: Set ObjFile = ObjFso.OpenTextFile(pupPackScreenFile)
+Dim DMDType: DMDType = 1'ObjFile.ReadLine
@@ -295 +295 @@
-'Dim PuPlayer
+Dim PuPlayer
@@ -321 +321 @@
-        'Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay")
+        Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay")
@@ -3090,3 +3090,3 @@
-    Dim WshShell:Set WshShell = CreateObject("WScript.Shell")
-    WshShell.RegWrite "HKCU\Software\UltraDMD\color", DMDOldColor, "REG_SZ"
-    WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor", DMDOldFullColor, "REG_SZ"
+    ' Dim WshShell:Set WshShell = CreateObject("WScript.Shell")
+    ' WshShell.RegWrite "HKCU\Software\UltraDMD\color", DMDOldColor, "REG_SZ"
+    ' WshShell.RegWrite "HKCU\Software\UltraDMD\fullcolor", DMDOldFullColor, "REG_SZ"
