--- Strikes And Spares (Bally 1978) 2.0.vbs.original
+++ Strikes And Spares (Bally 1978) 2.0.vbs
@@ -2095,0 +2096,26 @@
+Class StandupTarget
+  Private m_primary, m_prim, m_sw, m_animate
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public default Function init(primary, prim, sw, animate)
+    Set m_primary = primary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+
+    Set Init = Me
+  End Function
+End Class
+
+
@@ -2113,6 +2139,6 @@
-ST12 = Array(sw12, psw12, 12, 0)
-ST28 = Array(sw28, psw28, 28, 0)
-ST29 = Array(sw29, psw29, 29, 0)
-ST30 = Array(sw30, psw30, 30, 0)
-ST31 = Array(sw31, psw31, 31, 0)
-ST33 = Array(sw33, psw33, 33, 0)
+Set ST12 = (new StandupTarget)(sw12, psw12, 12, 0)
+Set ST28 = (new StandupTarget)(sw28, psw28, 28, 0)
+Set ST29 = (new StandupTarget)(sw29, psw29, 29, 0)
+Set ST30 = (new StandupTarget)(sw30, psw30, 30, 0)
+Set ST31 = (new StandupTarget)(sw31, psw31, 31, 0)
+Set ST33 = (new StandupTarget)(sw33, psw33, 33, 0)
@@ -2140 +2166 @@
-	STArray(i)(3) = STCheckHit(Activeball,STArray(i)(0))
+	STArray(i).animate = STCheckHit(Activeball,STArray(i).primary)
@@ -2142,2 +2168,2 @@
-	If STArray(i)(3) <> 0 Then
-		DTBallPhysics Activeball, STArray(i)(0).orientation, STMass
+	If STArray(i).animate <> 0 Then
+		DTBallPhysics Activeball, STArray(i).primary.orientation, STMass
@@ -2151 +2177 @@
-		If STArray(i)(2) = switch Then 
+		If STArray(i).sw = switch Then 
@@ -2182 +2208 @@
-		STArray(i)(3) = STAnimate(STArray(i)(0),STArray(i)(1),STArray(i)(2),STArray(i)(3))
+		STArray(i).animate = STAnimate(STArray(i).primary,STArray(i).prim,STArray(i).sw,STArray(i).animate)
