--- 1679379865_jurassicparklimitededition.vbs.original
+++ 1679379865_jurassicparklimitededition.vbs
@@ -493,9 +493,11 @@
 
 	Sub Table1_Init()
 		ChangeBall(ChooseBall)
-		resetbackglass
 		DMD_Init
 		LoadEM
+		InitPuP
+		calculatescore
+		resetbackglass
 		Dim i
 		Randomize
 		Loadhs
@@ -1400,6 +1402,22 @@
 	Const pAudio=7
 	Const pCallouts=8
 
+	Sub chilloutthemusic
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":10 }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":50 }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":10 }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":130 }"
+		vpmtimer.addtimer 3200, "turnitbackup'"
+	End Sub
+
+	Sub turnitbackup
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":"&(videovol)&" }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(sfxvol)&" }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":130 }"
+	End Sub	
+
+Sub InitPuP
 	if HasPuP Then
 	on error resume next
 	Set PuPlayer = CreateObject("PinUpPlayer.PinDisplay") 
@@ -1425,20 +1443,6 @@
 	PuPlayer.SetScreenex pCallouts,0,0,0,0,2
 	PuPlayer.hide pCallouts
 
-	Sub chilloutthemusic
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":10 }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":50 }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":10 }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":130 }"
-		vpmtimer.addtimer 3200, "turnitbackup'"
-	End Sub
-
-	Sub turnitbackup
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":"&(videovol)&" }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(sfxvol)&" }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":130 }"
-	End Sub
 
 
 	PuPlayer.playlistadd pAudio,"sfx", 1 , 0
@@ -1686,6 +1690,8 @@
 	PuPlayer.LabelNew pBackglass,"HighScoreL3",numberfont,	8,26316	,0,0,1,53,60,1,1
 	PuPlayer.LabelNew pBackglass,"HighScoreL4",numberfont,	5,26316	,0,1,1,50,65,1,1
 
+End Sub
+
 
 	Sub ruleshelperon
 		rulestime.enabled = 1
@@ -3658,7 +3664,7 @@
 	Dim point1:point1 = 1000
 	Dim bigpoints:bigpoints = 1
 
-	calculatescore
+	
 	Sub calculatescore
 	PuPlayer.LabelSet pBackglass,"lefttimer2",(bigpoints\10000) + 1 & "-" & point1\1000,1,"{'mt':2,'color':12971496, 'size': 4.8, 'xpos': 12.5, 'xalign': 1, 'ypos': 89.4, 'yalign': 1}"
 	PuPlayer.LabelSet pBackglass,"lefttimer",(bigpoints\10000) + 1 & "-" & point1\1000,1,"{'mt':2,'color':26316, 'size': 4.8, 'xpos': 12.5, 'xalign': 1, 'ypos': 89.2, 'yalign': 1}"
