--- 1679379865_jurassicparklimitededition.vbs.original
+++ 1679379865_jurassicparklimitededition.vbs
@@ -496 +495,0 @@
-		resetbackglass
@@ -498,0 +498,3 @@
+		InitPuP
+		calculatescore
+		resetbackglass
@@ -1402,0 +1405,16 @@
+	Sub chilloutthemusic
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":10 }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":50 }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":10 }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":130 }"
+		vpmtimer.addtimer 3200, "turnitbackup'"
+	End Sub
+
+	Sub turnitbackup
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":"&(videovol)&" }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(sfxvol)&" }"
+		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":130 }"
+	End Sub	
+
+Sub InitPuP
@@ -1428,14 +1445,0 @@
-	Sub chilloutthemusic
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":10 }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":50 }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":10 }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":130 }"
-		vpmtimer.addtimer 3200, "turnitbackup'"
-	End Sub
-
-	Sub turnitbackup
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 2, ""FN"":11, ""VL"":"&(videovol)&" }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 4, ""FN"":11, ""VL"":"&(soundtrackvol)&" }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 7, ""FN"":11, ""VL"":"&(sfxvol)&" }"
-		PuPlayer.SendMSG "{ ""mt"":301, ""SN"": 8, ""FN"":11, ""VL"":130 }"
-	End Sub
@@ -1688,0 +1693,2 @@
+End Sub
+
@@ -3661 +3667 @@
-	calculatescore
+	
