--- Fireball XL5.vbs.original
+++ Fireball XL5.vbs
@@ -2427 +2427 @@
-				FlexDMD.Stage.GetImage("Game").SetSize (DMD_VideoCounter - 64)*2, (DMD_VideoCounter - 64)/2
+				FlexDMD.Stage.GetImage("Game").SetSize 2 * (DMD_VideoCounter - 64), (DMD_VideoCounter - 64)/2
@@ -3104 +3104 @@
-				FlexDMD.Stage.GetImage("Game").SetSize (DMD_VideoCounter - 64)*2, (DMD_VideoCounter - 64)/2
+				FlexDMD.Stage.GetImage("Game").SetSize 2 * (DMD_VideoCounter - 64), (DMD_VideoCounter - 64)/2
