--- Diner (Williams 1990) VPW Mod 1.0.2.vbs.original
+++ Diner (Williams 1990) VPW Mod 1.0.2.vbs
@@ -4178,0 +4179,33 @@
+Class DropTarget
+  Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+  Public Property Let Secondary(input): Set m_secondary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+  Public Property Let IsDropped(input): m_isDropped = input: End Property
+
+  Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+    Set m_primary = primary
+    Set m_secondary = secondary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+    m_isDropped = isDropped
+
+    Set Init = Me
+  End Function
+End Class
+
@@ -4196,0 +4230 @@
+'       isDropped:  Boolean which determines whether a drop target is dropped. Set to false if they are initially raised, true if initially dropped.
@@ -4198,6 +4232,6 @@
-DT1 = Array(rootbeer, rootbeer_a, rootbeer_p, swRootBeer, 0)
-DT2 = Array(fries, fries_a, fries_p, swFries, 0)
-DT3 = Array(icedtea, icedtea_a, icedtea_p, swIcedTea, 0)
-DT4 = Array(hotdog, hotdog_a, hotdog_p, swHotdog, 0)
-DT5 = Array(burger, burger_a, burger_p, swBurger, 0)
-DT6 = Array(chili, chili_a, chili_p, swChili, 0)
+Set DT1 = (new DropTarget)(rootbeer, rootbeer_a, rootbeer_p, swRootBeer, 0, false)
+Set DT2 = (new DropTarget)(fries, fries_a, fries_p, swFries, 0, false)
+Set DT3 = (new DropTarget)(icedtea, icedtea_a, icedtea_p, swIcedTea, 0, false)
+Set DT4 = (new DropTarget)(hotdog, hotdog_a, hotdog_p, swHotdog, 0, false)
+Set DT5 = (new DropTarget)(burger, burger_a, burger_p, swBurger, 0, false)
+Set DT6 = (new DropTarget)(chili, chili_a, chili_p, swChili, 0, false)
@@ -4236,3 +4270,3 @@
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
@@ -4247 +4281 @@
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
@@ -4255 +4289 @@
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
@@ -4262 +4296 @@
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
@@ -4317 +4351 @@
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
