--- Aztec High-Tapped (Williams 1976).vbs.original
+++ Aztec High-Tapped (Williams 1976).vbs
@@ -1747,6 +1747,7 @@
 '*******************************************************************************************
 
 '*********** ROLLING SOUND *********************************
+Const tnob = 1 ' total number of balls for this table (at the same time)
 ReDim rolling(tnob)
 InitRolling
 Sub InitRolling:Dim i:For i=0 to (tnob-1):rolling(i) = False:Next:End Sub
@@ -1780,7 +1781,6 @@
 
 
 '*********** BALL SHADOW *********************************
-Const tnob = 1 ' total number of balls for this table (at the same time)
 ReDim BallShadow(tnob-1)
 InitBallShadow
 
