--- "Mr Black (Taito do Brasil - 1984) v4.vbs.original"	2023-10-30 11:56:41.381252152 +0100
+++ "Mr Black (Taito do Brasil - 1984) v4.vbs"	2023-10-30 11:52:16.018294506 +0100
@@ -61,7 +61,7 @@
 '************
 
 Sub table1_Init
-	NVramPatchLoad
+	'NVramPatchLoad
     vpmInit me
     With Controller
         .GameName = cGameName
@@ -145,7 +145,10 @@
 
 Sub table1_Paused:Controller.Pause = 1:End Sub
 Sub table1_unPaused:Controller.Pause = 0:End Sub
-Sub table1_exit:NVramPatchExit:Controller.stop:End Sub
+Sub table1_exit
+    'NVramPatchExit
+    Controller.stop
+End Sub
 
 '**********
 ' Keys
@@ -510,7 +513,7 @@
     End If
     UpdateLeds
     UpdateLamps
-    NVramPatchKeyCheck
+    'NVramPatchKeyCheck
 End Sub
 
 
