--- Iron Maiden (Original 2022) VPW 1.0.12.vbs.original
+++ Iron Maiden (Original 2022) VPW 1.0.12.vbs
@@ -7959,0 +7960,33 @@
+Class DropTarget
+  Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+  Public Property Let Secondary(input): Set m_secondary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+  Public Property Let IsDropped(input): m_isDropped = input: End Property
+
+  Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+    Set m_primary = primary
+    Set m_secondary = secondary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+    m_isDropped = isDropped
+
+    Set Init = Me
+  End Function
+End Class
+
@@ -7979,3 +8012,3 @@
-DT1 = Array(Target004, Target004a, Target004p, 1, 0)
-DT2 = Array(Target005, Target005a, Target005p, 2, 0)
-DT3 = Array(Target006, Target006a, Target006p, 3, 0)
+Set DT1 = (new DropTarget)(Target004, Target004a, Target004p, 1, 0, false)
+Set DT2 = (new DropTarget)(Target005, Target005a, Target005p, 2, 0, false)
+Set DT3 = (new DropTarget)(Target006, Target006a, Target006p, 3, 0, false)
@@ -8015,2 +8048,2 @@
-'WriteToLog "     ", "DTArray(i)(2).transz:" & DTArray(i)(2).transz & " " & DTArray(i)(4)
-'	if DTArray(i)(2).transz = -DTDropUnits or DTArray(i)(4)>=1 then 	
+'WriteToLog "     ", "DTArray(i).prim.transz:" & DTArray(i).prim.transz & " " & DTArray(i).animate
+'	if DTArray(i).prim.transz = -DTDropUnits or DTArray(i).animate>=1 then 	
@@ -8030,3 +8063,3 @@
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
@@ -8034 +8067 @@
-'WriteToLog "     ", "DTHit:(" & i & ") " & DTArray(i)(4)
+'WriteToLog "     ", "DTHit:(" & i & ") " & DTArray(i).animate
@@ -8045 +8078 @@
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
@@ -8054 +8087 @@
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
@@ -8061 +8094 @@
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
@@ -8120 +8153 @@
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
@@ -8363 +8396,2 @@
-    ExecuteGlobal "Sub " & MyLamp.Name & "_Timer:" & "SetLamp (" & MyLamp.Name & ".UserValue - INT(" & MyLamp.Name & ".UserValue))*100, " & MyLamp.Name & ".UserValue MOD 2:" & MyLamp.Name & ".UserValue= " & MyLamp.Name & ".UserValue -1:If " & MyLamp.Name & ".UserValue < 0 then " & MyLamp.Name & ".TimerEnabled=0:End If:End Sub"
+
+    ExecuteGlobal "Sub " & MyLamp.Name & "_Timer:" & "SetLamp 100*(" & MyLamp.Name & ".UserValue - INT(" & MyLamp.Name & ".UserValue)), " & MyLamp.Name & ".UserValue MOD 2:" & MyLamp.Name & ".UserValue= " & MyLamp.Name & ".UserValue -1:If " & MyLamp.Name & ".UserValue < 0 then " & MyLamp.Name & ".TimerEnabled=0:End If:End Sub"
@@ -8604,2 +8638,2 @@
-	Execute "set vL=v" & n.name
-	Execute "set pL=p" & n.name
+	Set vL = eval("v" & n.name)
+	Set pL = eval("p" & n.name)
@@ -8687,0 +8722,5 @@
+	If IsEmpty(vL) OR IsEmpty(pL) Then
+		'debug.print "SetLightColorRGB: v" & n.name & " = " & TypeName(vL) & " p" & n.name & " = " & TypeName(pL)
+		exit sub
+	End If
+
@@ -14022 +14061 @@
-			CheckDrops (tgtIndex+1) Mod 3
+			CheckDrops ((tgtIndex+1) Mod 3)
