--- TimeLord3.0.4.vbs.original	2023-06-07 15:45:30
+++ TimeLord3.0.4.vbs	2023-06-07 15:49:36
@@ -4068,7 +4068,7 @@
 	ResetNewBallLights
 	bFlippersEnabled = True
 '	vpmtimer.addtimer 500, "StopDragon'"
-	If GloryTableOn=True Then GloryTableOn=False: 	ResetStartofGameVariables():TurnOnStartOfGameLights:GloryStopAtEndOfBallComplete: Else
+	If GloryTableOn=True Then GloryTableOn=False: 	ResetStartofGameVariables():TurnOnStartOfGameLights:GloryStopAtEndOfBallComplete
 	vpmtimer.addtimer 500, "ResetModesAndAwards'"
 End Sub
 
@@ -5454,7 +5454,7 @@
 'CheckWormHoleJackpots
 '**************************
 Sub CheckWormHoleJackpots
-	If l24.State=2 Then:l23F.State=0:l23G.State=2 Else
+	If l24.State=2 Then:l23F.State=0:l23G.State=2
 	LS_MajorAward.UpdateInterval = 10
 	LS_MajorAward.Play SeqUpOn, 35, 2
 	vpmTimer.AddTimer 2000, "WormLocksActivate'"
@@ -6370,7 +6370,7 @@
 'Start TimeWarp Jackpots and disable Reverse time until 30 second timer runs out
 
 Sub CheckEnableTimeWarpJackpots
-	If	TimeLordComplete=True Then: DoNothing: Else
+	If	TimeLordComplete=True Then DoNothing
 	StartTimeWarpJackPots
 	
 End Sub	
@@ -6429,9 +6429,9 @@
 
 
 Sub CheckSpiralRampAward
-	If lGlory1.State=2 Then:GloryBillions:StartFlasherSpiralAward:Else
-	If l61.State=2 and GloryTableOn=False Then:TimeWarpJackPotCheck: StartFlasherSpiralAward:Else
-	If l61B.State=2 and GloryTableOn=False and TWJPFiring=False Then: AwardDefyGravity:Else
+	If lGlory1.State=2 Then GloryBillions:StartFlasherSpiralAward
+	If l61.State=2 and GloryTableOn=False Then:TimeWarpJackPotCheck: StartFlasherSpiralAward
+	If l61B.State=2 and GloryTableOn=False and TWJPFiring=False Then: AwardDefyGravity
 	If l61A.State=2 and GloryTableOn=False and TWJPFiring=False Then: AwardSpaceWalk
 End Sub
 
@@ -6527,8 +6527,8 @@
 Sub TimeWarpJackPotCheck
 debug.print "Trigger10_Hit"
 	PlaySoundAt "RampOut1_6",Trigger10
-	If l62C.State=2 Then:l62A.State=2:l62C.State=0:TimeWarpSuperJackPot:PlaySound "CO_SuperJackPot":Else
-	If l62B.State=2 Then:l62C.State=2:l62B.State=0:PlaySound "CO_JackPot":TimeWarpJackPot2:Else
+	If l62C.State=2 Then:l62A.State=2:l62C.State=0:TimeWarpSuperJackPot:PlaySound "CO_SuperJackPot"
+	If l62B.State=2 Then:l62C.State=2:l62B.State=0:PlaySound "CO_JackPot":TimeWarpJackPot2
 	If l62A.State=2 Then:l62B.State=2:l62A.State=0:PlaySound "CO_JackPot":TimeWarpJackPot2
 End Sub
 
@@ -6586,8 +6586,8 @@
 	vpmtimer.addtimer 500, "StartClockHandsClockwise'"
 	TWJPFiring=False
 	TimeWarpEasyTimer.Enabled=False
-	If  l21B.State=1 Then:l21.State=2:l20.State=1:l19.State=1:Else
-	If  l20A.State=1 Then:l20.State=2:l21.State=0:l19.State=1:Else
+	If  l21B.State=1 Then:l21.State=2:l20.State=1:l19.State=1
+	If  l20A.State=1 Then:l20.State=2:l21.State=0:l19.State=1
 	If  l19A.State=1 Then:l19.State=2:l20.State=0:l21.State=0
 End Sub
 
@@ -6598,8 +6598,8 @@
 	vpmtimer.addtimer 500, "StartClockHandsClockwise'"
 	TWJPFiring=False
 	TimeWarpEasyTimer.Enabled=False
-	If  l21B.State=1 Then:l21.State=2:l20.State=1:l19.State=1:Else
-	If  l20A.State=1 Then:l20.State=2:l21.State=0:l19.State=1:Else
+	If  l21B.State=1 Then:l21.State=2:l20.State=1:l19.State=1
+	If  l20A.State=1 Then:l20.State=2:l21.State=0:l19.State=1
 	If  l19A.State=1 Then:l19.State=2:l20.State=0:l21.State=0
 End Sub
 
