--- ./Cyclone (Williams 1988) 2.0.1/Cyclone (Williams 1988) 2.0.1.vbs.original	2023-06-02 16:49:52
+++ ./Cyclone (Williams 1988) 2.0.1/Cyclone (Williams 1988) 2.0.1.vbs	2023-06-02 16:50:07
@@ -5,6 +5,7 @@
 Option Explicit
 Randomize
 
+
 On Error Resume Next
 ExecuteGlobal GetTextFile("controller.vbs")
 If Err Then MsgBox "Can't open controller.vbs"
\ No newline at end of file
@@ -22,8 +23,9 @@
 Const SFlipperOn = "FlipperUp"
 Const SFlipperOff = "FlipperDown"
 Const SCoin = "Coin"
-Const BSize = 25
-Const BMass = 1.7
+
+const BallSize = 50
+const BallMass = 1.7
 
 '**************************************************************
 '                    USER OPTIONS
\ No newline at end of file
