--- Thunderbirds original 2022 v1.0.2.vbs.original
+++ Thunderbirds original 2022 v1.0.2.vbs
@@ -117,0 +118 @@
+	FlexONPlayfield = false
@@ -1824 +1825 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text1", "TB4text T.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text1", "TB4text T.png", 88, 110, 2900
@@ -1826 +1827 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text4", "TB4text T4.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text4", "TB4text T4.png", 88, 110, 2900
@@ -1828 +1829 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text5", "TB4text TB.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text5", "TB4text TB.png", 88, 110, 2900
@@ -1830 +1831 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text7", "TB4text TB4.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text7", "TB4text TB4.png", 88, 110, 2900
@@ -1850 +1851 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text2", "TB4text B.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text2", "TB4text B.png", 88, 110, 2900
@@ -1852 +1853 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text6", "TB4text B4.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text6", "TB4text B4.png", 88, 110, 2900
@@ -1854 +1855 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text5", "TB4text TB.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text5", "TB4text TB.png", 88, 110, 2900
@@ -1856 +1857 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text7", "TB4text TB4.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text7", "TB4text TB4.png", 88, 110, 2900
@@ -1876 +1877 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text3", "TB4text 4.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text3", "TB4text 4.png", 88, 110, 2900
@@ -1878 +1879 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text6", "TB4text B4.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text6", "TB4text B4.png", 88, 110, 2900
@@ -1880 +1881 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text4", "TB4text T4.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text4", "TB4text T4.png", 88, 110, 2900
@@ -1882 +1883 @@
-			createsplash "TB4", "TB4VID", "tb4.gif", "TB4text7", "TB4text TB4.png", 88, 110, 2900
+			createsplash "TB4", "TB4VID", "TB4.gif", "TB4text7", "TB4text TB4.png", 88, 110, 2900
@@ -2129 +2130 @@
-			createsplash "TB2", "TB2VID1", "tb2 launch 2.gif", "TB2text1", "TB2 - T.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID1", "TB2 launch 2.gif", "TB2text1", "TB2 - T.png", 100, 140, 3000
@@ -2131 +2132 @@
-			createsplash "TB2", "TB2VID2", "tb2 launch 3.gif", "TB2text4", "TB2 - T2.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID2", "TB2 launch 3.gif", "TB2text4", "TB2 - T2.png", 100, 140, 3000
@@ -2133 +2134 @@
-			createsplash "TB2", "TB2VID2", "tb2 launch 3.gif", "TB2text5", "TB2 - TB.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID2", "TB2 launch 3.gif", "TB2text5", "TB2 - TB.png", 100, 140, 3000
@@ -2135 +2136 @@
-			createsplash "TB2", "TB2VID3", "tb2 launch 4.gif", "TB2text7", "TB2 - TB2.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID3", "TB2 launch 4.gif", "TB2text7", "TB2 - TB2.png", 100, 140, 3000
@@ -2151 +2152 @@
-			createsplash "TB2", "TB2VID1", "tb2 launch 2.gif", "TB2text2", "TB2 - B.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID1", "TB2 launch 2.gif", "TB2text2", "TB2 - B.png", 100, 140, 3000
@@ -2153 +2154 @@
-			createsplash "TB2", "TB2VID2", "tb2 launch 3.gif", "TB2text6", "TB2 - B2.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID2", "TB2 launch 3.gif", "TB2text6", "TB2 - B2.png", 100, 140, 3000
@@ -2155 +2156 @@
-			createsplash "TB2", "TB2VID2", "tb2 launch 3.gif", "TB2text5", "TB2 - TB.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID2", "TB2 launch 3.gif", "TB2text5", "TB2 - TB.png", 100, 140, 3000
@@ -2157 +2158 @@
-			createsplash "TB2", "TB2VID3", "tb2 launch 4.gif", "TB2text7", "TB2 - TB2.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID3", "TB2 launch 4.gif", "TB2text7", "TB2 - TB2.png", 100, 140, 3000
@@ -2172 +2173 @@
-			createsplash "TB2", "TB2VID1", "tb2 launch 2.gif", "TB2text3", "TB2 - 2.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID1", "TB2 launch 2.gif", "TB2text3", "TB2 - 2.png", 100, 140, 3000
@@ -2174 +2175 @@
-			createsplash "TB2", "TB2VID2", "tb2 launch 3.gif", "TB2text6", "TB2 - B2.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID2", "TB2 launch 3.gif", "TB2text6", "TB2 - B2.png", 100, 140, 3000
@@ -2176 +2177 @@
-			createsplash "TB2", "TB2VID2", "tb2 launch 3.gif", "TB2text4", "TB2 - T2.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID2", "TB2 launch 3.gif", "TB2text4", "TB2 - T2.png", 100, 140, 3000
@@ -2178 +2179 @@
-			createsplash "TB2", "TB2VID3", "tb2 launch 4.gif", "TB2text7", "TB2 - TB2.png", 100, 140, 3000
+			createsplash "TB2", "TB2VID3", "TB2 launch 4.gif", "TB2text7", "TB2 - TB2.png", 100, 140, 3000
