--- Wheel of Fortune (Stern 2007) 1.0.vbs.original
+++ Wheel of Fortune (Stern 2007) 1.0.vbs
@@ -12,0 +13,2 @@
+Const cGameName = "wof_500"
+
@@ -46,2 +47,0 @@
-Const cGameName = "wof_500"
-
@@ -570,2 +570,2 @@
-Dim LampState(400), FadingLevel(400)
-Dim FlashSpeedUp(400), FlashSpeedDown(400), FlashMin(400), FlashMax(400), FlashLevel(400)
+Dim LampState(610), FadingLevel(610)
+Dim FlashSpeedUp(610), FlashSpeedDown(610), FlashMin(610), FlashMax(610), FlashLevel(610)
@@ -916 +916 @@
-	Debug.Print Vol
+	'Debug.Print Vol
