--- "./AC-DC LUCI Premium VR (Stern 2013) v1.1.vbs"	2024-01-16 18:53:32.398400000 +0100
+++ "./AC-DC LUCI Premium VR (Stern 2013) v1.1.vbs.dmd"	2023-12-23 15:26:43.105370513 +0100
@@ -184,11 +184,14 @@
 Dim tableheight: tableheight = ACDC.height
 Const UseVPMModSol = True
 
-Dim UseVPMDMD,CustomDMD,DesktopMode,CabinetMode
+'Dim UseVPMDMD,CustomDMD,DesktopMode,CabinetMode
+Dim UseVPMColoredDMD,CustomDMD,DesktopMode,CabinetMode
+
 DesktopMode = ACDC.ShowDT : CustomDMD = False
 If Right(cGamename,1)="c" Then CustomDMD= False
-If CustomDMD OR (NOT DesktopMode AND NOT CustomDMD) Then UseVPMDMD = False		'hides the internal VPMDMD when using the color ROM or when table is in Full Screen and color ROM is not in use
-If DesktopMode AND NOT CustomDMD Then UseVPMDMD = True							'shows the internal VPMDMD when in desktop mode and color ROM is not in use
+'If CustomDMD OR (NOT DesktopMode AND NOT CustomDMD) Then UseVPMDMD = False		'hides the internal VPMDMD when using the color ROM or when If CustomDMD OR (NOT DesktopMode AND NOT CustomDMD) Then UseVPMColoredDMD = False		'hides the internal VPMDMD when using the color ROM or when table is in Full Screen and color ROM is not in use
+'If DesktopMode AND NOT CustomDMD Then UseVPMDMD = True							'shows the internal VPMDMD when in desktop mode and color ROM
+If DesktopMode AND NOT CustomDMD Then UseVPMColoredDMD = True							'shows the internal VPMDMD when in desktop mode and color ROM is not in use
 Scoretext.visible = NOT CustomDMD												'hides the textbox when using the color ROM
 
 LoadVPM "02800000", "Sam.VBS", 3.54
