--- Maverick (Data East 1994) VPW v1.3.vbs.original
+++ Maverick (Data East 1994) VPW v1.3.vbs
@@ -850,0 +851,33 @@
+Class DropTarget
+  Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+  Public Property Let Secondary(input): Set m_secondary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+  Public Property Let IsDropped(input): m_isDropped = input: End Property
+
+  Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+    Set m_primary = primary
+    Set m_secondary = secondary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+    m_isDropped = isDropped
+
+    Set Init = Me
+  End Function
+End Class
+
@@ -868,0 +902 @@
+'       isDropped:  Boolean which determines whether a drop target is dropped. Set to false if they are initially raised, true if initially dropped
@@ -871,20 +905,20 @@
-DT22 = Array(sw22, sw22offset, primsw22, 22, 0)
-DT23 = Array(sw23, sw23offset, primsw23, 23, 0)
-DT24 = Array(sw24, sw24offset, primsw24, 24, 0)
-
-DT25 = Array(sw25, sw25offset, primsw25, 25, 0)
-DT26 = Array(sw26, sw26offset, primsw26, 26, 0)
-DT27 = Array(sw27, sw27offset, primsw27, 27, 0)
-DT28 = Array(sw28, sw28offset, primsw28, 28, 0)
-DT29 = Array(sw29, sw29offset, primsw29, 29, 0)
-
-DT17 = Array(sw17, sw17offset, primsw17, 17, 0)
-DT18 = Array(sw18, sw18offset, primsw18, 18, 0)
-DT19 = Array(sw19, sw19offset, primsw19, 19, 0)
-DT20 = Array(sw20, sw20offset, primsw20, 20, 0)
-DT21 = Array(sw21, sw21offset, primsw21, 21, 0)
-
-DT33 = Array(sw33, sw33offset, primsw33, 33, 0)
-DT34 = Array(sw34, sw34offset, primsw34, 34, 0)
-DT35 = Array(sw35, sw35offset, primsw35, 35, 0)
-DT36 = Array(sw36, sw36offset, primsw36, 36, 0)
+Set DT22 = (new DropTarget)(sw22, sw22offset, primsw22, 22, 0, false)
+Set DT23 = (new DropTarget)(sw23, sw23offset, primsw23, 23, 0, false)
+Set DT24 = (new DropTarget)(sw24, sw24offset, primsw24, 24, 0, false)
+
+Set DT25 = (new DropTarget)(sw25, sw25offset, primsw25, 25, 0, false)
+Set DT26 = (new DropTarget)(sw26, sw26offset, primsw26, 26, 0, false)
+Set DT27 = (new DropTarget)(sw27, sw27offset, primsw27, 27, 0, false)
+Set DT28 = (new DropTarget)(sw28, sw28offset, primsw28, 28, 0, false)
+Set DT29 = (new DropTarget)(sw29, sw29offset, primsw29, 29, 0, false)
+
+Set DT17 = (new DropTarget)(sw17, sw17offset, primsw17, 17, 0, false)
+Set DT18 = (new DropTarget)(sw18, sw18offset, primsw18, 18, 0, false)
+Set DT19 = (new DropTarget)(sw19, sw19offset, primsw19, 19, 0, false)
+Set DT20 = (new DropTarget)(sw20, sw20offset, primsw20, 20, 0, false)
+Set DT21 = (new DropTarget)(sw21, sw21offset, primsw21, 21, 0, false)
+
+Set DT33 = (new DropTarget)(sw33, sw33offset, primsw33, 33, 0, false)
+Set DT34 = (new DropTarget)(sw34, sw34offset, primsw34, 34, 0, false)
+Set DT35 = (new DropTarget)(sw35, sw35offset, primsw35, 35, 0, false)
+Set DT36 = (new DropTarget)(sw36, sw36offset, primsw36, 36, 0, false)
@@ -923,3 +957,3 @@
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
@@ -934 +968 @@
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
@@ -942 +976 @@
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
@@ -949 +983 @@
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
@@ -1004 +1038 @@
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
