--- ./Lord of the Rings (Stern 2003)_VPW_v1.3_10.6/Lord of the Rings (Stern 2003)_VPW_v1.3_10.6.vbs.original	2023-06-04 22:12:04
+++ ./Lord of the Rings (Stern 2003)_VPW_v1.3_10.6/Lord of the Rings (Stern 2003)_VPW_v1.3_10.6.vbs	2023-06-08 13:19:01
@@ -3164,9 +3164,9 @@
 	Private Sub Class_Initialize : redim ballvel(0) : redim ballvelx(0): redim ballvely(0) : End Sub 
 
 	Public Sub Update()        'tracks in-ball-velocity
-		dim str, b, highestID
+		dim str, b, AllBalls, highestID : allBalls = getballs
 
-		for each b in gBOT
+		for each b in allballs
 			if b.id >= HighestID then highestID = b.id
 		Next
 
\ No newline at end of file
@@ -3174,7 +3174,7 @@
 		if uBound(ballvelx) < highestID then redim ballvelx(highestID)        'set bounds
 		if uBound(ballvely) < highestID then redim ballvely(highestID)        'set bounds
 
-		for each b in gBOT
+		for each b in allballs
 			ballvel(b.id) = BallSpeed(b)
 			ballvelx(b.id) = b.velx
 			ballvely(b.id) = b.vely
\ No newline at end of file
@@ -4322,12 +4322,12 @@
 		End If
 		if x = uBound(RampBalls) then 	'debug
 			'Debug.print "WireRampOn error, ball queue is full: " & vbnewline & RampBalls(0, 0) & vbnewline & _
-				Typename(RampBalls(1, 0)) & " ID:" & RampBalls(1, 1) & "type:" & RampType(1) & vbnewline & _
-				Typename(RampBalls(2, 0)) & " ID:" & RampBalls(2, 1) & "type:" & RampType(2) & vbnewline & _
-				Typename(RampBalls(3, 0)) & " ID:" & RampBalls(3, 1) & "type:" & RampType(3) & vbnewline & _
-				Typename(RampBalls(4, 0)) & " ID:" & RampBalls(4, 1) & "type:" & RampType(4) & vbnewline & _
-				Typename(RampBalls(5, 0)) & " ID:" & RampBalls(5, 1) & "type:" & RampType(5) & vbnewline & _
-				" "
+				'Typename(RampBalls(1, 0)) & " ID:" & RampBalls(1, 1) & "type:" & RampType(1) & vbnewline & _
+				'Typename(RampBalls(2, 0)) & " ID:" & RampBalls(2, 1) & "type:" & RampType(2) & vbnewline & _
+				'Typename(RampBalls(3, 0)) & " ID:" & RampBalls(3, 1) & "type:" & RampType(3) & vbnewline & _
+				'Typename(RampBalls(4, 0)) & " ID:" & RampBalls(4, 1) & "type:" & RampType(4) & vbnewline & _
+				'Typename(RampBalls(5, 0)) & " ID:" & RampBalls(5, 1) & "type:" & RampType(5) & vbnewline & _
+				'" "
 		End If
 	next
 End Sub
\ No newline at end of file
