--- MrDoom (Recel 1979)1.3.0.vbs.original
+++ MrDoom (Recel 1979)1.3.0.vbs
@@ -1838,0 +1839,33 @@
+Class DropTarget
+  Private m_primary, m_secondary, m_prim, m_sw, m_animate, m_isDropped
+
+  Public Property Get Primary(): Set Primary = m_primary: End Property
+  Public Property Let Primary(input): Set m_primary = input: End Property
+
+  Public Property Get Secondary(): Set Secondary = m_secondary: End Property
+  Public Property Let Secondary(input): Set m_secondary = input: End Property
+
+  Public Property Get Prim(): Set Prim = m_prim: End Property
+  Public Property Let Prim(input): Set m_prim = input: End Property
+
+  Public Property Get Sw(): Sw = m_sw: End Property
+  Public Property Let Sw(input): m_sw = input: End Property
+
+  Public Property Get Animate(): Animate = m_animate: End Property
+  Public Property Let Animate(input): m_animate = input: End Property
+
+  Public Property Get IsDropped(): IsDropped = m_isDropped: End Property
+  Public Property Let IsDropped(input): m_isDropped = input: End Property
+
+  Public default Function init(primary, secondary, prim, sw, animate, isDropped)
+    Set m_primary = primary
+    Set m_secondary = secondary
+    Set m_prim = prim
+    m_sw = sw
+    m_animate = animate
+    m_isDropped = isDropped
+
+    Set Init = Me
+  End Function
+End Class
+
@@ -1859,3 +1892,3 @@
-DT31 = Array(sw31, sw31y, psw31, 31, 0)
-DT32 = Array(sw32, sw32y, psw32, 32, 0)
-DT33 = Array(sw33, sw33y, psw33, 33, 0)
+Set DT31 = (new DropTarget)(sw31, sw31y, psw31, 31, 0, false)
+Set DT32 = (new DropTarget)(sw32, sw32y, psw32, 32, 0, false)
+Set DT33 = (new DropTarget)(sw33, sw33y, psw33, 33, 0, false)
@@ -1864,3 +1897,3 @@
-DT28 = Array(sw28, sw28y, psw28, 28, 0)
-DT29 = Array(sw29, sw29y, psw29, 29, 0)
-DT30 = Array(sw30, sw30y, psw30, 30, 0)
+Set DT28 = (new DropTarget)(sw28, sw28y, psw28, 28, 0, false)
+Set DT29 = (new DropTarget)(sw29, sw29y, psw29, 29, 0, false)
+Set DT30 = (new DropTarget)(sw30, sw30y, psw30, 30, 0, false)
@@ -1903,3 +1936,3 @@
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
@@ -1916 +1949 @@
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
@@ -1924 +1957 @@
-	DTArray(i)(4) = 1
+	DTArray(i).animate = 1
@@ -1931 +1964 @@
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
@@ -1986 +2019 @@
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
