--- Heavy Metal Meltdown (Bally 1987).vbs.original
+++ Heavy Metal Meltdown (Bally 1987).vbs
@@ -12,7 +12,7 @@
 
 '******************* Options *********************
 ' DMD/Backglass Controller Setting
-Const cController = 1		'0=Use value defined in cController.txt, 1=VPinMAME, 2=UVP server, 3=B2S server
+Const cController = 3		'0=Use value defined in cController.txt, 1=VPinMAME, 2=UVP server, 3=B2S server
 '*************************************************
 
 '*************************************************************
