--- Stingray (Stern 1977).vbs.original
+++ Stingray (Stern 1977).vbs
@@ -1699,5 +1699,5 @@
-DT25 = Array(sw25, sw25a, sw25p, 25, 0, False)
-DT26 = Array(sw26, sw26a, sw26p, 26, 0, False)
-DT27 = Array(sw27, sw27a, sw27p, 27, 0, False)
-DT28 = Array(sw28, sw28a, sw28p, 28, 0, False)
-DT29 = Array(sw29, sw29a, sw29p, 29, 0, False)
+Set DT25 = (new DropTarget)(sw25, sw25a, sw25p, 25, 0, false)
+Set DT26 = (new DropTarget)(sw26, sw26a, sw26p, 26, 0, false)
+Set DT27 = (new DropTarget)(sw27, sw27a, sw27p, 27, 0, false)
+Set DT28 = (new DropTarget)(sw28, sw28a, sw28p, 28, 0, false)
+Set DT29 = (new DropTarget)(sw29, sw29a, sw29p, 29, 0, false)
@@ -1734,3 +1734,3 @@
-	DTArray(i)(4) =  DTCheckBrick(Activeball,DTArray(i)(2))
-	If DTArray(i)(4) = 1 or DTArray(i)(4) = 3 or DTArray(i)(4) = 4 Then
-		DTBallPhysics Activeball, DTArray(i)(2).rotz, DTMass
+	DTArray(i).animate =  DTCheckBrick(Activeball,DTArray(i).prim)
+	If DTArray(i).animate = 1 or DTArray(i).animate = 3 or DTArray(i).animate = 4 Then
+		DTBallPhysics Activeball, DTArray(i).prim.rotz, DTMass
@@ -1745 +1745 @@
-	DTArray(i)(4) = -1
+	DTArray(i).animate = -1
@@ -1753 +1753 @@
-	DTArray(i)(4) = 1
+	DTArray(i).animate  = 1
@@ -1760 +1760 @@
-		If DTArray(i)(3) = switch Then DTArrayID = i:Exit Function 
+		If DTArray(i).sw = switch Then DTArrayID = i:Exit Function 
@@ -1815 +1815 @@
-		DTArray(i)(4) = DTAnimate(DTArray(i)(0),DTArray(i)(1),DTArray(i)(2),DTArray(i)(3),DTArray(i)(4))
+		DTArray(i).animate = DTAnimate(DTArray(i).primary,DTArray(i).secondary,DTArray(i).prim,DTArray(i).sw,DTArray(i).animate)
@@ -1871 +1871 @@
-			DTArray(ind)(5) = true 'Mark target as dropped
+			DTArray(ind).isDropped = true 'Mark target as dropped
@@ -1926 +1926 @@
-		DTArray(ind)(5) = false 'Mark target as not dropped
+		DTArray(ind).isDropped = false 'Mark target as not dropped
